# GSEA research notes - HEAD-KIDNEY

## 4 WPC

### DNA vaccine

| Description                                                  | NES       | setSize | Count |
| ------------------------------------------------------------ | --------- | ------- | ----- |
| Peptide ligand-binding receptors                             | -1.898369 | 37      | 16    |
| Class A/1 (Rhodopsin-like receptors)                         | -1.868515 | 79      | 31    |
| GPCR ligand binding                                          | -1.851556 | 101     | 37    |
| Cellular hexose transport                                    | -1.717279 | 11      | 5     |
| Sensory processing of sound by inner hair cells of the cochlea | -1.714307 | 33      | 15    |
| Signaling by GPCR                                            | -1.686628 | 258     | 85    |
| Sensory processing of sound                                  | -1.674182 | 36      | 17    |
| RORA activates gene expression                               | -1.668492 | 16      | 5     |
| Constitutive Signaling by Aberrant PI3K in Cancer            | -1.667587 | 35      | 14    |
| Nitric oxide stimulates guanylate cyclase                    | -1.665922 | 10      | 5     |
| Glutamate Neurotransmitter Release Cycle                     | -1.641438 | 15      | 10    |
| HS-GAG biosynthesis                                          | -1.634413 | 18      | 8     |
| Long-term potentiation                                       | -1.619079 | 10      | 5     |
| IRS-related events triggered by IGF1R                        | -1.601692 | 28      | 12    |
| GPCR downstream signalling                                   | -1.596801 | 241     | 78    |
| Receptor-type tyrosine-protein phosphatases                  | -1.588085 | 12      | 5     |
| Sensory processing of sound by outer hair cells of the cochlea | -1.587560 | 25      | 14    |
| NCAM1 interactions                                           | -1.586362 | 16      | 6     |
| G alpha (12/13) signalling events                            | -1.580873 | 53      | 34    |
| Metal ion SLC transporters                                   | -1.576531 | 14      | 7     |
| NRAGE signals death through JNK                              | -1.572897 | 43      | 16    |
| Class B/2 (Secretin family receptors)                        | -1.551199 | 21      | 6     |
| Signaling by Type 1 Insulin-like Growth Factor 1 Receptor (IGF1R) | -1.550559 | 29      | 14    |
| IGF1R signaling cascade                                      | -1.550559 | 29      | 14    |
| G alpha (i) signalling events                                | -1.548316 | 100     | 35    |
| Negative regulation of the PI3K/AKT network                  | -1.546372 | 62      | 22    |
| Potassium Channels                                           | -1.545506 | 23      | 9     |
| Dopamine Neurotransmitter Release Cycle                      | -1.544575 | 14      | 6     |
| Metabolism of fat-soluble vitamins                           | -1.543244 | 25      | 12    |
| Activation of SMO                                            | -1.540024 | 11      | 6     |
| Signaling by WNT in cancer                                   | -1.515090 | 21      | 10    |
| Laminin interactions                                         | -1.507777 | 18      | 5     |
| RHOB GTPase cycle                                            | -1.502878 | 64      | 27    |
| NCAM signaling for neurite out-growth                        | -1.499037 | 33      | 10    |
| NGF-stimulated transcription                                 | -1.496005 | 24      | 11    |
| Disassembly of the destruction complex and recruitment of AXIN to the membrane | -1.492778 | 20      | 10    |
| Transport of bile salts and organic acids, metal ions and amine compounds | -1.485597 | 37      | 17    |
| Defective B3GALT6 causes EDSP2 and SEMDJL1                   | -1.481914 | 10      | 4     |
| CDC42 GTPase cycle                                           | -1.480730 | 125     | 60    |
| SLC-mediated transmembrane transport                         | -1.470681 | 129     | 53    |
| Sensory Perception                                           | -1.469184 | 80      | 30    |
| Ca2+ pathway                                                 | -1.459038 | 34      | 11    |
| Protein-protein interactions at synapses                     | -1.457523 | 43      | 14    |
| Muscle contraction                                           | -1.443707 | 90      | 24    |
| Signaling by NTRKs                                           | -1.421794 | 98      | 38    |
| Sphingolipid metabolism                                      | -1.417192 | 70      | 11    |
| G alpha (q) signalling events                                | -1.414677 | 88      | 44    |
| RHOA GTPase cycle                                            | -1.409192 | 127     | 48    |
| PI3K/AKT Signaling in Cancer                                 | -1.406538 | 57      | 20    |
| Signaling by NTRK1 (TRKA)                                    | -1.380517 | 84      | 34    |
| MyD88:MAL(TIRAP) cascade initiated on plasma membrane        | -1.379232 | 78      | 34    |
| Toll Like Receptor TLR1:TLR2 Cascade                         | -1.379232 | 78      | 34    |
| Toll Like Receptor TLR6:TLR2 Cascade                         | -1.379232 | 78      | 34    |
| Toll Like Receptor 2 (TLR2) Cascade                          | -1.379232 | 78      | 34    |
| Neuronal System                                              | -1.361198 | 164     | 57    |
| Transmission across Chemical Synapses                        | -1.353636 | 117     | 40    |
| Axon guidance                                                | 1.138802  | 361     | 94    |
| Adaptive Immune System                                       | 1.201348  | 488     | 108   |
| Neutrophil degranulation                                     | 1.208304  | 296     | 116   |
| SARS-CoV Infections                                          | 1.311292  | 305     | 94    |
| RHO GTPases Activate Formins                                 | 1.336346  | 97      | 45    |
| SARS-CoV-2 Infection                                         | 1.346521  | 214     | 68    |
| UCH proteinases                                              | 1.378611  | 75      | 24    |
| SARS-CoV-1 Infection                                         | 1.391961  | 111     | 44    |
| CLEC7A (Dectin-1) signaling                                  | 1.407009  | 81      | 20    |
| Downstream TCR signaling                                     | 1.412984  | 70      | 22    |
| KEAP1-NFE2L2 pathway                                         | 1.416750  | 102     | 23    |
| RNA Polymerase II Pre-transcription Events                   | 1.441413  | 71      | 20    |
| Global Genome Nucleotide Excision Repair (GG-NER)            | 1.450598  | 70      | 18    |
| ER-Phagosome pathway                                         | 1.450672  | 63      | 23    |
| Transport of Mature mRNAs Derived from Intronless Transcripts | 1.454109  | 35      | 9     |
| Mitotic Prometaphase                                         | 1.456858  | 153     | 61    |
| Protein localization                                         | 1.459801  | 121     | 34    |
| Neddylation                                                  | 1.460006  | 193     | 46    |
| SUMOylation of chromatin organization proteins               | 1.470953  | 47      | 12    |
| Formation of paraxial mesoderm                               | 1.473598  | 54      | 17    |
| PTEN Regulation                                              | 1.476454  | 117     | 28    |
| RHO GTPase Effectors                                         | 1.481098  | 198     | 76    |
| Antigen processing-Cross presentation                        | 1.481510  | 75      | 29    |
| C-type lectin receptors (CLRs)                               | 1.483194  | 95      | 23    |
| EML4 and NUDC in mitotic spindle formation                   | 1.487638  | 84      | 42    |
| Transport of Mature mRNA Derived from an Intronless Transcript | 1.488959  | 34      | 9     |
| Nucleotide Excision Repair                                   | 1.490695  | 91      | 23    |
| ISG15 antiviral mechanism                                    | 1.498335  | 55      | 12    |
| Export of Viral Ribonucleoproteins from Nucleus              | 1.502679  | 27      | 8     |
| NEP/NS2 Interacts with the Cellular Export Machinery         | 1.502679  | 27      | 8     |
| Translesion synthesis by Y family DNA polymerases bypasses lesions on DNA template | 1.503871  | 32      | 8     |
| Transport of the SLBP independent Mature mRNA                | 1.506275  | 28      | 8     |
| Glycolysis                                                   | 1.512998  | 51      | 15    |
| HCMV Early Events                                            | 1.516398  | 54      | 15    |
| Mitotic Spindle Checkpoint                                   | 1.521303  | 90      | 40    |
| Amplification of signal from the kinetochores                | 1.522766  | 76      | 36    |
| Amplification  of signal from unattached  kinetochores via a MAD2  inhibitory signal | 1.522766  | 76      | 36    |
| ATF4 activates genes in response to endoplasmic reticulum  stress | 1.529761  | 22      | 9     |
| Nuclear Envelope (NE) Reassembly                             | 1.534652  | 51      | 24    |
| TCR signaling                                                | 1.538692  | 87      | 28    |
| SARS-CoV-2-host interactions                                 | 1.540857  | 139     | 58    |
| Cellular response to chemical stress                         | 1.542184  | 159     | 54    |
| mRNA Splicing                                                | 1.543553  | 180     | 73    |
| BBSome-mediated cargo-targeting to cilium                    | 1.546516  | 15      | 7     |
| Mitotic G2-G2/M phases                                       | 1.546591  | 150     | 48    |
| Signaling by the B Cell Receptor (BCR)                       | 1.547450  | 93      | 25    |
| Transcription of the HIV genome                              | 1.547464  | 61      | 19    |
| Transcriptional regulation by RUNX1                          | 1.549167  | 142     | 32    |
| G2/M Transition                                              | 1.552014  | 149     | 48    |
| Vpr-mediated nuclear import of PICs                          | 1.552132  | 26      | 8     |
| Somitogenesis                                                | 1.554455  | 46      | 17    |
| Deposition of new CENPA-containing nucleosomes at the centromere | 1.554581  | 21      | 6     |
| Nucleosome assembly                                          | 1.554581  | 21      | 6     |
| Regulation of Glucokinase by Glucokinase Regulatory Protein  | 1.563185  | 25      | 8     |
| Defective TPR may confer susceptibility towards thyroid papillary carcinoma (TPC) | 1.563185  | 25      | 8     |
| Metabolism of amino acids and derivatives                    | 1.563329  | 267     | 112   |
| mRNA Splicing - Minor Pathway                                | 1.563560  | 44      | 12    |
| Gene and protein expression by JAK-STAT signaling after Interleukin-12 stimulation | 1.563972  | 27      | 12    |
| Meiotic synapsis                                             | 1.564023  | 23      | 5     |
| SUMOylation of RNA binding proteins                          | 1.565725  | 38      | 11    |
| mRNA 3'-end processing                                       | 1.570448  | 51      | 27    |
| HIV Transcription Initiation                                 | 1.570483  | 40      | 11    |
| RNA Polymerase II HIV Promoter Escape                        | 1.570483  | 40      | 11    |
| RNA Polymerase II Promoter Escape                            | 1.570483  | 40      | 11    |
| RNA Polymerase II Transcription Pre-Initiation And Promoter Opening | 1.570483  | 40      | 11    |
| RNA Polymerase II Transcription Initiation                   | 1.570483  | 40      | 11    |
| RNA Polymerase II Transcription Initiation And Promoter Clearance | 1.570483  | 40      | 11    |
| Signaling by FGFR2 IIIa TM                                   | 1.576421  | 16      | 6     |
| TRAF6 mediated IRF7 activation in TLR7/8 or 9 signaling      | 1.580540  | 10      | 8     |
| mRNA Splicing - Major Pathway                                | 1.581028  | 173     | 71    |
| Termination of translesion DNA synthesis                     | 1.582349  | 25      | 7     |
| Transport of Mature mRNA derived from an Intron-Containing Transcript | 1.587545  | 61      | 29    |
| mRNA decay by 3' to 5' exoribonuclease                       | 1.596144  | 14      | 7     |
| Telomere Maintenance                                         | 1.596961  | 62      | 16    |
| APC/C:Cdc20 mediated degradation of Cyclin B                 | 1.602951  | 20      | 6     |
| Resolution of Sister Chromatid Cohesion                      | 1.603552  | 94      | 45    |
| FCERI mediated NF-kB activation                              | 1.610498  | 65      | 21    |
| Mitochondrial protein import                                 | 1.611419  | 50      | 24    |
| Dual Incision in GG-NER                                      | 1.612127  | 33      | 8     |
| Removal of the Flap Intermediate from the C-strand           | 1.617365  | 14      | 10    |
| SUMOylation of DNA replication proteins                      | 1.617774  | 39      | 13    |
| Downstream signaling events of B Cell Receptor (BCR)         | 1.621439  | 68      | 23    |
| Transport of Mature Transcript to Cytoplasm                  | 1.626752  | 69      | 33    |
| MHC class II antigen presentation                            | 1.629213  | 78      | 19    |
| Processing of Capped Intron-Containing Pre-mRNA              | 1.631444  | 239     | 99    |
| NS1 Mediated Effects on Host Pathways                        | 1.632671  | 32      | 10    |
| MAPK6/MAPK4 signaling                                        | 1.639220  | 76      | 22    |
| SUMOylation of SUMOylation proteins                          | 1.640954  | 29      | 9     |
| Gap-filling DNA repair synthesis and ligation in TC-NER      | 1.641227  | 53      | 13    |
| Hedgehog 'off' state                                         | 1.643923  | 81      | 27    |
| HIV elongation arrest and recovery                           | 1.644714  | 30      | 9     |
| Pausing and recovery of HIV elongation                       | 1.644714  | 30      | 9     |
| Chromosome Maintenance                                       | 1.645756  | 80      | 28    |
| PERK regulates gene expression                               | 1.652403  | 27      | 11    |
| Pausing and recovery of Tat-mediated HIV elongation          | 1.653135  | 29      | 9     |
| Tat-mediated HIV elongation arrest and recovery              | 1.653135  | 29      | 9     |
| Interconversion of nucleotide di- and triphosphates          | 1.655394  | 19      | 8     |
| Regulation of PTEN stability and activity                    | 1.660899  | 59      | 19    |
| tRNA processing in the nucleus                               | 1.662955  | 48      | 13    |
| DNA Damage Bypass                                            | 1.663316  | 41      | 12    |
| Base Excision Repair                                         | 1.671815  | 37      | 22    |
| Cell Cycle, Mitotic                                          | 1.674286  | 407     | 163   |
| Formation of TC-NER Pre-Incision Complex                     | 1.675714  | 45      | 11    |
| Dual incision in TC-NER                                      | 1.678389  | 53      | 13    |
| SARS-CoV-1-host interactions                                 | 1.679226  | 74      | 34    |
| Interactions of Vpr with host cellular proteins              | 1.679614  | 27      | 9     |
| Formation of tubulin folding intermediates by CCT/TriC       | 1.680516  | 12      | 10    |
| Late Phase of HIV Life Cycle                                 | 1.683110  | 115     | 37    |
| Prefoldin mediated transfer of substrate  to CCT/TriC        | 1.688304  | 18      | 11    |
| Cooperation of Prefoldin and TriC/CCT  in actin and tubulin folding | 1.688304  | 18      | 11    |
| rRNA processing                                              | 1.688988  | 172     | 58    |
| Condensation of Prometaphase Chromosomes                     | 1.691717  | 11      | 5     |
| Translesion synthesis by POLI                                | 1.693945  | 15      | 4     |
| M Phase                                                      | 1.695185  | 287     | 119   |
| RNA Polymerase II Transcription Termination                  | 1.703843  | 56      | 30    |
| Translesion Synthesis by POLH                                | 1.709257  | 17      | 5     |
| Disorders of transmembrane transporters                      | 1.709713  | 116     | 32    |
| SUMOylation of ubiquitinylation proteins                     | 1.720380  | 31      | 10    |
| SUMOylation of DNA damage response and repair proteins       | 1.721382  | 61      | 16    |
| Regulation of mRNA stability by proteins that bind AU-rich elements | 1.721465  | 75      | 26    |
| Collagen degradation                                         | 1.722414  | 31      | 10    |
| snRNP Assembly                                               | 1.726941  | 44      | 13    |
| Metabolism of non-coding RNA                                 | 1.726941  | 44      | 13    |
| Gap-filling DNA repair synthesis and ligation in GG-NER      | 1.728069  | 20      | 8     |
| Unwinding of DNA                                             | 1.730626  | 11      | 3     |
| Nuclear events mediated by NFE2L2                            | 1.732387  | 76      | 31    |
| ABC-family proteins mediated transport                       | 1.735827  | 76      | 25    |
| Regulation of RUNX2 expression and activity                  | 1.739000  | 56      | 19    |
| Cell Cycle Checkpoints                                       | 1.741388  | 215     | 98    |
| Translesion synthesis by REV1                                | 1.741781  | 14      | 4     |
| Condensation of Prophase Chromosomes                         | 1.742712  | 16      | 10    |
| Maternal to zygotic transition (MZT)                         | 1.746421  | 42      | 3     |
| Activation of NF-kappaB in B cells                           | 1.752844  | 57      | 20    |
| TNFR2 non-canonical NF-kB pathway                            | 1.755333  | 60      | 21    |
| Lagging Strand Synthesis                                     | 1.757589  | 18      | 8     |
| Metabolic disorders of biological oxidation enzymes          | 1.763286  | 17      | 6     |
| HIV Life Cycle                                               | 1.764963  | 123     | 41    |
| Abortive elongation of HIV-1 transcript in the absence of Tat | 1.767302  | 22      | 12    |
| Transcription-Coupled Nucleotide Excision Repair (TC-NER)    | 1.769134  | 65      | 19    |
| Resolution of AP sites via the multiple-nucleotide patch replacement pathway | 1.772279  | 21      | 11    |
| Glutathione conjugation                                      | 1.792277  | 17      | 10    |
| G1/S DNA Damage Checkpoints                                  | 1.793583  | 58      | 21    |
| ATF6 (ATF6-alpha) activates chaperone genes                  | 1.802080  | 10      | 7     |
| Hh mutants abrogate ligand secretion                         | 1.803321  | 50      | 20    |
| p53-Dependent G1 DNA Damage Response                         | 1.806012  | 57      | 21    |
| p53-Dependent G1/S DNA damage checkpoint                     | 1.806012  | 57      | 21    |
| Signaling by NOTCH4                                          | 1.807572  | 66      | 23    |
| rRNA processing in the nucleus and cytosol                   | 1.808687  | 162     | 57    |
| Cyclin A:Cdk2-associated events at S phase entry             | 1.813637  | 76      | 25    |
| Inhibition of DNA recombination at telomere                  | 1.818177  | 21      | 7     |
| Mitochondrial translation termination                        | 1.832632  | 76      | 43    |
| Cyclin E associated events during G1/S transition            | 1.835220  | 74      | 25    |
| G2/M Checkpoints                                             | 1.836895  | 115     | 55    |
| Maturation of TCA enzymes and regulation of TCA cycle        | 1.837153  | 17      | 12    |
| Transcriptional regulation by small RNAs                     | 1.842060  | 45      | 14    |
| Mitochondrial translation initiation                         | 1.842166  | 76      | 44    |
| Mitotic Anaphase                                             | 1.844995  | 184     | 80    |
| Mitotic Metaphase and Anaphase                               | 1.849734  | 185     | 81    |
| Hedgehog ligand biogenesis                                   | 1.852418  | 52      | 21    |
| PCNA-Dependent Long Patch Base Excision Repair               | 1.871621  | 17      | 8     |
| Host Interactions of HIV factors                             | 1.874978  | 107     | 36    |
| Complex I biogenesis                                         | 1.875018  | 44      | 22    |
| Mitochondrial translation                                    | 1.876080  | 82      | 46    |
| Mitochondrial translation elongation                         | 1.887312  | 76      | 45    |
| Dectin-1 mediated noncanonical NF-kB signaling               | 1.895239  | 52      | 19    |
| HIV Infection                                                | 1.896702  | 191     | 64    |
| Heme biosynthesis                                            | 1.899694  | 11      | 7     |
| RUNX1 regulates transcription of genes involved in differentiation of HSCs | 1.901211  | 65      | 22    |
| Viral Messenger RNA Synthesis                                | 1.905425  | 38      | 14    |
| ATF6 (ATF6-alpha) activates chaperones                       | 1.907050  | 12      | 8     |
| Major pathway of rRNA processing in the nucleolus and cytosol | 1.907547  | 152     | 55    |
| AUF1 (hnRNP D0) binds and destabilizes mRNA                  | 1.912492  | 47      | 18    |
| FBXL7 down-regulates AURKA during mitotic entry and in early mitosis | 1.921740  | 48      | 19    |
| FGFR2 alternative splicing                                   | 1.923406  | 21      | 10    |
| Cellular response to starvation                              | 1.923815  | 125     | 41    |
| Degradation of GLI1 by the proteasome                        | 1.925014  | 50      | 20    |
| PCP/CE pathway                                               | 1.925900  | 70      | 37    |
| APC/C-mediated degradation of cell cycle proteins            | 1.927213  | 77      | 28    |
| Regulation of mitotic cell cycle                             | 1.927213  | 77      | 28    |
| Metabolism of polyamines                                     | 1.938700  | 53      | 27    |
| Activation of the pre-replicative complex                    | 1.941105  | 28      | 18    |
| Cellular response to hypoxia                                 | 1.941721  | 65      | 23    |
| NIK-->noncanonical NF-kB signaling                           | 1.945799  | 51      | 19    |
| Activation of Matrix Metalloproteinases                      | 1.954516  | 11      | 4     |
| GLI3 is processed to GLI3R by the proteasome                 | 1.968317  | 51      | 20    |
| Activation of ATR in response to replication stress          | 1.970960  | 33      | 21    |
| Switching of origins to a post-replicative state             | 1.978475  | 81      | 41    |
| Citric acid cycle (TCA cycle)                                | 1.981681  | 30      | 18    |
| SCF-beta-TrCP mediated degradation of Emi1                   | 1.984259  | 48      | 19    |
| Asymmetric localization of PCP proteins                      | 1.984539  | 52      | 28    |
| Mitotic G1 phase and G1/S transition                         | 1.988280  | 125     | 62    |
| Separation of Sister Chromatids                              | 1.988696  | 148     | 74    |
| DNA strand elongation                                        | 1.994820  | 29      | 11    |
| Degradation of GLI2 by the proteasome                        | 1.997713  | 50      | 20    |
| Stabilization of p53                                         | 2.004577  | 48      | 18    |
| ABC transporter disorders                                    | 2.006541  | 61      | 24    |
| Cristae formation                                            | 2.019671  | 24      | 17    |
| Regulation of APC/C activators between G1/S and early anaphase | 2.029764  | 71      | 27    |
| Nonsense-Mediated Decay (NMD)                                | 2.031480  | 92      | 48    |
| Nonsense Mediated Decay (NMD) enhanced by the Exon Junction Complex (EJC) | 2.031480  | 92      | 48    |
| Vpu mediated degradation of CD4                              | 2.031771  | 45      | 19    |
| RHO GTPases activate PKNs                                    | 2.032475  | 30      | 18    |
| SCF(Skp2)-mediated degradation of p27/p21                    | 2.045367  | 54      | 22    |
| Orc1 removal from chromatin                                  | 2.046792  | 64      | 36    |
| Negative regulation of NOTCH4 signaling                      | 2.048350  | 47      | 20    |
| Recognition of DNA damage by PCNA-containing replication complex | 2.062587  | 25      | 11    |
| GSK3B and BTRC:CUL1-mediated-degradation of NFE2L2           | 2.062993  | 46      | 19    |
| Degradation of DVL                                           | 2.076107  | 50      | 29    |
| G1/S Transition                                              | 2.078545  | 113     | 60    |
| S Phase                                                      | 2.082303  | 144     | 46    |
| Respiratory electron transport                               | 2.088733  | 83      | 41    |
| APC:Cdc20 mediated degradation of cell cycle proteins prior to satisfation of the cell cycle checkpoint | 2.090415  | 65      | 24    |
| Autodegradation of the E3 ubiquitin ligase COP1              | 2.097057  | 46      | 18    |
| Aerobic respiration and respiratory electron transport       | 2.113940  | 161     | 74    |
| Formation of ATP by chemiosmotic coupling                    | 2.114518  | 12      | 7     |
| APC/C:Cdc20 mediated degradation of mitotic proteins         | 2.121036  | 66      | 25    |
| Selenoamino acid metabolism                                  | 2.125163  | 95      | 52    |
| Regulation of ornithine decarboxylase (ODC)                  | 2.130723  | 46      | 25    |
| Translation                                                  | 2.132676  | 249     | 124   |
| Oxygen-dependent proline hydroxylation of Hypoxia-inducible Factor Alpha | 2.149238  | 58      | 21    |
| Activation of APC/C and APC/C:Cdc20 mediated degradation of mitotic proteins | 2.152249  | 67      | 26    |
| Degradation of AXIN                                          | 2.168457  | 48      | 19    |
| Respiratory electron transport, ATP synthesis by chemiosmotic coupling, and heat production by uncoupling proteins. | 2.169438  | 98      | 48    |
| Cross-presentation of soluble exogenous antigens (endosomes) | 2.169850  | 42      | 25    |
| SARS-CoV-1 modulates host translation machinery              | 2.171885  | 30      | 17    |
| Cdc20:Phospho-APC/C mediated degradation of Cyclin A         | 2.179124  | 64      | 33    |
| CDK-mediated phosphorylation and removal of Cdc6             | 2.189034  | 64      | 33    |
| Ubiquitin Mediated Degradation of Phosphorylated Cdc25A      | 2.195298  | 45      | 18    |
| p53-Independent DNA Damage Response                          | 2.195298  | 45      | 18    |
| p53-Independent G1/S DNA damage checkpoint                   | 2.195298  | 45      | 18    |
| The role of GTSE1 in G2/M progression after G2 checkpoint    | 2.200940  | 58      | 25    |
| APC/C:Cdc20 mediated degradation of Securin                  | 2.201093  | 58      | 22    |
| APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1 | 2.205780  | 64      | 24    |
| Assembly of the pre-replicative complex                      | 2.206283  | 76      | 27    |
| Regulation of activated PAK-2p34 by proteasome mediated degradation | 2.211546  | 44      | 18    |
| Selenocysteine synthesis                                     | 2.220933  | 75      | 52    |
| Ubiquitin-dependent degradation of Cyclin D                  | 2.222888  | 46      | 19    |
| Metabolism of porphyrins                                     | 2.223918  | 17      | 11    |
| Regulation of Apoptosis                                      | 2.232589  | 47      | 27    |
| Autodegradation of Cdh1 by Cdh1:APC/C                        | 2.244973  | 56      | 21    |
| Eukaryotic Translation Termination                           | 2.252142  | 74      | 53    |
| Regulation of RUNX3 expression and activity                  | 2.253519  | 47      | 18    |
| Synthesis of DNA                                             | 2.253831  | 106     | 40    |
| Influenza Infection                                          | 2.259532  | 126     | 70    |
| Vif-mediated degradation of APOBEC3G                         | 2.266014  | 47      | 20    |
| Detoxification of Reactive Oxygen Species                    | 2.266782  | 22      | 18    |
| DNA Replication                                              | 2.269363  | 115     | 62    |
| SARS-CoV-2 modulates host translation machinery              | 2.287955  | 39      | 21    |
| Influenza Viral RNA Transcription and Replication            | 2.298649  | 111     | 61    |
| Hh mutants are degraded by ERAD                              | 2.304187  | 48      | 28    |
| Peptide chain elongation                                     | 2.307492  | 71      | 52    |
| DNA Replication Pre-Initiation                               | 2.321567  | 90      | 49    |
| Activation of the mRNA upon binding of the cap-binding complex and eIFs, and subsequent binding to 43S | 2.323919  | 51      | 24    |
| Eukaryotic Translation Elongation                            | 2.335797  | 73      | 53    |
| Translation initiation complex formation                     | 2.343264  | 50      | 24    |
| Response of EIF2AK4 (GCN2) to amino acid deficiency          | 2.344863  | 82      | 46    |
| Defective CFTR causes cystic fibrosis                        | 2.364787  | 52      | 31    |
| Ribosomal scanning and start codon recognition               | 2.389005  | 51      | 25    |
| Formation of the ternary complex, and subsequently, the 43S complex | 2.391335  | 45      | 24    |
| Signaling by ROBO receptors                                  | 2.439710  | 166     | 95    |
| Nonsense Mediated Decay (NMD) independent of the Exon Junction Complex (EJC) | 2.440381  | 75      | 53    |
| Viral mRNA Translation                                       | 2.456060  | 70      | 52    |
| SRP-dependent cotranslational protein targeting to membrane  | 2.459457  | 89      | 52    |
| Eukaryotic Translation Initiation                            | 2.477005  | 98      | 66    |
| Cap-dependent Translation Initiation                         | 2.477005  | 98      | 66    |
| L13a-mediated translational silencing of Ceruloplasmin expression | 2.513564  | 91      | 55    |
| Formation of a pool of free 40S subunits                     | 2.519407  | 83      | 61    |
| GTP hydrolysis and joining of the 60S ribosomal subunit      | 2.535719  | 92      | 56    |
| Regulation of expression of SLITs and ROBOs                  | 2.575939  | 136     | 83    |
| Mitochondrial protein degradation                            | 2.577030  | 81      | 50    |

### EOMES

| Description                                                  | NES       | setSize | Count |
| ------------------------------------------------------------ | --------- | ------- | ----- |
| Sensory processing of sound by inner hair cells of the cochlea | -1.748837 | 31      | 17    |
| Long-term potentiation                                       | -1.688954 | 10      | 5     |
| Dopamine Neurotransmitter Release Cycle                      | -1.664041 | 13      | 7     |
| Sensory processing of sound                                  | -1.663711 | 34      | 16    |
| Protein-protein interactions at synapses                     | -1.581498 | 42      | 15    |
| Transmission across Chemical Synapses                        | -1.479524 | 115     | 34    |
| Metabolism of carbohydrates                                  | 1.578805  | 189     | 49    |
| Response to elevated platelet cytosolic Ca2+                 | 1.581291  | 78      | 21    |
| Metabolism of steroids                                       | 1.584888  | 97      | 24    |
| rRNA processing                                              | 1.593360  | 172     | 64    |
| Diseases of metabolism                                       | 1.595154  | 144     | 35    |
| Platelet degranulation                                       | 1.596438  | 74      | 20    |
| Extracellular matrix organization                            | 1.600723  | 155     | 29    |
| Glucose metabolism                                           | 1.634594  | 54      | 14    |
| Disorders of transmembrane transporters                      | 1.636374  | 114     | 33    |
| rRNA processing in the nucleus and cytosol                   | 1.664360  | 162     | 75    |
| Translation                                                  | 1.667454  | 249     | 105   |
| Maternal to zygotic transition (MZT)                         | 1.700397  | 40      | 4     |
| Pyruvate metabolism                                          | 1.705549  | 37      | 14    |
| Cellular response to starvation                              | 1.706554  | 124     | 57    |
| Degradation of the extracellular matrix                      | 1.718058  | 66      | 14    |
| Glycogen metabolism                                          | 1.732991  | 20      | 8     |
| Plasma lipoprotein assembly, remodeling, and clearance       | 1.733256  | 46      | 15    |
| Collagen biosynthesis and modifying enzymes                  | 1.735078  | 30      | 15    |
| Metabolism of nucleotides                                    | 1.742821  | 74      | 21    |
| Nucleotide catabolism                                        | 1.743800  | 24      | 7     |
| Formation of ATP by chemiosmotic coupling                    | 1.747353  | 12      | 11    |
| Amyloid fiber formation                                      | 1.750055  | 34      | 13    |
| Lysine catabolism                                            | 1.756481  | 11      | 7     |
| Peroxisomal lipid metabolism                                 | 1.757965  | 19      | 12    |
| Metabolism of vitamins and cofactors                         | 1.758470  | 131     | 35    |
| Activation of Matrix Metalloproteinases                      | 1.758975  | 11      | 5     |
| Arachidonic acid metabolism                                  | 1.759124  | 30      | 14    |
| Phase II - Conjugation of compounds                          | 1.784450  | 48      | 16    |
| Assembly of active LPL and LIPC lipase complexes             | 1.800929  | 11      | 5     |
| Major pathway of rRNA processing in the nucleolus and cytosol | 1.801319  | 152     | 73    |
| Formation of Fibrin Clot (Clotting Cascade)                  | 1.802891  | 13      | 3     |
| Fatty acid metabolism                                        | 1.807310  | 118     | 43    |
| Integrin cell surface interactions                           | 1.809870  | 47      | 11    |
| Metabolism of folate and pterines                            | 1.831727  | 11      | 6     |
| Degradation of cysteine and homocysteine                     | 1.840771  | 13      | 8     |
| SARS-CoV-2 modulates host translation machinery              | 1.845231  | 39      | 26    |
| SARS-CoV-1 modulates host translation machinery              | 1.851809  | 30      | 21    |
| Attachment and Entry                                         | 1.864010  | 11      | 7     |
| Protein localization                                         | 1.884595  | 121     | 39    |
| Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell | 1.888422  | 27      | 10    |
| Respiratory electron transport                               | 1.896364  | 83      | 41    |
| Metabolism of porphyrins                                     | 1.897188  | 17      | 11    |
| Sulfur amino acid metabolism                                 | 1.897366  | 24      | 12    |
| Binding and Uptake of Ligands by Scavenger Receptors         | 1.907952  | 23      | 7     |
| Metabolic disorders of biological oxidation enzymes          | 1.925527  | 17      | 7     |
| Nonsense-Mediated Decay (NMD)                                | 1.928087  | 92      | 56    |
| Nonsense Mediated Decay (NMD) enhanced by the Exon Junction Complex (EJC) | 1.928087  | 92      | 56    |
| Heme biosynthesis                                            | 1.951164  | 11      | 8     |
| Plasma lipoprotein remodeling                                | 1.955769  | 19      | 9     |
| Translation initiation complex formation                     | 1.964003  | 50      | 26    |
| Post-translational protein phosphorylation                   | 1.966010  | 50      | 18    |
| Scavenging by Class A Receptors                              | 1.980234  | 14      | 5     |
| Activation of the mRNA upon binding of the cap-binding complex and eIFs, and subsequent binding to 43S | 1.992614  | 51      | 27    |
| Cytochrome P450 - arranged by substrate type                 | 1.993397  | 22      | 10    |
| Peroxisomal protein import                                   | 1.994364  | 44      | 18    |
| Detoxification of Reactive Oxygen Species                    | 2.005465  | 22      | 13    |
| Ribosomal scanning and start codon recognition               | 2.007552  | 51      | 27    |
| Respiratory electron transport, ATP synthesis by chemiosmotic coupling, and heat production by uncoupling proteins. | 2.015502  | 98      | 50    |
| Formation of the ternary complex, and subsequently, the 43S complex | 2.025754  | 45      | 26    |
| Metabolism of fat-soluble vitamins                           | 2.035588  | 25      | 12    |
| Influenza Infection                                          | 2.037178  | 125     | 67    |
| Influenza Viral RNA Transcription and Replication            | 2.066656  | 110     | 64    |
| Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs) | 2.092458  | 56      | 21    |
| Aerobic respiration and respiratory electron transport       | 2.110440  | 160     | 78    |
| Retinoid metabolism and transport                            | 2.147202  | 21      | 9     |
| Glutathione conjugation                                      | 2.149258  | 17      | 7     |
| Collagen degradation                                         | 2.163898  | 30      | 11    |
| Signaling by ROBO receptors                                  | 2.178178  | 165     | 91    |
| Eukaryotic Translation Termination                           | 2.188410  | 74      | 51    |
| Selenoamino acid metabolism                                  | 2.193095  | 95      | 61    |
| Regulation of expression of SLITs and ROBOs                  | 2.201981  | 135     | 78    |
| Selenocysteine synthesis                                     | 2.230963  | 75      | 50    |
| Visual phototransduction                                     | 2.236597  | 36      | 15    |
| Peptide chain elongation                                     | 2.250277  | 71      | 50    |
| SRP-dependent cotranslational protein targeting to membrane  | 2.268162  | 89      | 60    |
| Eukaryotic Translation Elongation                            | 2.279521  | 73      | 51    |
| Nonsense Mediated Decay (NMD) independent of the Exon Junction Complex (EJC) | 2.287740  | 75      | 51    |
| Response of EIF2AK4 (GCN2) to amino acid deficiency          | 2.295444  | 82      | 53    |
| Mitochondrial protein degradation                            | 2.316663  | 81      | 46    |
| Eukaryotic Translation Initiation                            | 2.333654  | 98      | 59    |
| Cap-dependent Translation Initiation                         | 2.333654  | 98      | 59    |
| Viral mRNA Translation                                       | 2.342171  | 70      | 50    |
| L13a-mediated translational silencing of Ceruloplasmin expression | 2.359260  | 91      | 56    |
| GTP hydrolysis and joining of the 60S ribosomal subunit      | 2.378197  | 92      | 57    |
| Metabolism of amino acids and derivatives                    | 2.381380  | 265     | 125   |
| Formation of a pool of free 40S subunits                     | 2.401536  | 83      | 56    |
| Phase I - Functionalization of compounds                     | 2.407338  | 41      | 18    |
| Biological oxidations                                        | 2.417072  | 94      | 36    |

### GATA3

| Description                                                  | NES       | setSize | Count |
| ------------------------------------------------------------ | --------- | ------- | ----- |
| Synthesis of very long-chain fatty acyl-CoAs                 | -1.684479 | 19      | 9     |
| Sensory processing of sound                                  | -1.668433 | 32      | 14    |
| Sensory processing of sound by inner hair cells of the cochlea | -1.629033 | 29      | 15    |
| HS-GAG biosynthesis                                          | -1.626025 | 17      | 8     |
| Sensory processing of sound by outer hair cells of the cochlea | -1.608837 | 21      | 9     |
| Metabolism of fat-soluble vitamins                           | -1.606457 | 25      | 16    |
| Nicotinamide salvaging                                       | -1.595758 | 13      | 7     |
| Plasma lipoprotein remodeling                                | -1.594360 | 19      | 11    |
| Bile acid and bile salt metabolism                           | -1.591882 | 22      | 9     |
| Synthesis of PIPs at the late endosome membrane              | -1.553295 | 11      | 6     |
| RHOB GTPase cycle                                            | -1.541174 | 63      | 21    |
| NRAGE signals death through JNK                              | -1.535744 | 42      | 19    |
| Peptide ligand-binding receptors                             | -1.517745 | 34      | 12    |
| Class A/1 (Rhodopsin-like receptors)                         | -1.513317 | 74      | 31    |
| G alpha (12/13) signalling events                            | -1.508970 | 52      | 33    |
| Muscle contraction                                           | -1.501913 | 88      | 21    |
| Signaling by GPCR                                            | -1.499479 | 245     | 105   |
| GPCR ligand binding                                          | -1.484694 | 91      | 33    |
| GPCR downstream signalling                                   | -1.476687 | 230     | 98    |
| Metabolism of vitamins and cofactors                         | -1.474144 | 131     | 48    |
| SLC-mediated transmembrane transport                         | -1.441127 | 127     | 34    |
| G alpha (q) signalling events                                | -1.429814 | 86      | 40    |
| Metabolism of lipids                                         | -1.257625 | 500     | 139   |
| Neutrophil degranulation                                     | 1.283297  | 293     | 96    |
| Nervous system development                                   | 1.287317  | 363     | 112   |
| SARS-CoV Infections                                          | 1.300935  | 303     | 82    |
| Axon guidance                                                | 1.317153  | 350     | 110   |
| SARS-CoV-2-host interactions                                 | 1.342103  | 138     | 40    |
| Signaling by NOTCH                                           | 1.349235  | 139     | 43    |
| Adaptive Immune System                                       | 1.351711  | 481     | 135   |
| Mitotic Prometaphase                                         | 1.363991  | 151     | 46    |
| Cellular response to chemical stress                         | 1.377313  | 159     | 52    |
| G2/M Checkpoints                                             | 1.403760  | 115     | 35    |
| mRNA Splicing                                                | 1.416873  | 179     | 64    |
| CLEC7A (Dectin-1) signaling                                  | 1.428114  | 80      | 26    |
| Cell Cycle Checkpoints                                       | 1.435963  | 215     | 70    |
| Mitotic G2-G2/M phases                                       | 1.443498  | 148     | 54    |
| G2/M Transition                                              | 1.450079  | 147     | 54    |
| Amplification of signal from the kinetochores                | 1.461076  | 76      | 27    |
| Amplification  of signal from unattached  kinetochores via a MAD2  inhibitory signal | 1.461076  | 76      | 27    |
| Cell Cycle                                                   | 1.462536  | 497     | 149   |
| Metabolism of amino acids and derivatives                    | 1.476012  | 264     | 106   |
| Switching of origins to a post-replicative state             | 1.480159  | 81      | 31    |
| ABC-family proteins mediated transport                       | 1.482829  | 74      | 27    |
| rRNA processing                                              | 1.489591  | 172     | 63    |
| mRNA Splicing - Major Pathway                                | 1.505290  | 172     | 64    |
| Regulation of TNFR1 signaling                                | 1.508306  | 37      | 11    |
| FCERI mediated NF-kB activation                              | 1.511784  | 65      | 24    |
| UCH proteinases                                              | 1.515172  | 75      | 28    |
| Processing of Capped Intron-Containing Pre-mRNA              | 1.522370  | 236     | 84    |
| Hedgehog 'off' state                                         | 1.526536  | 80      | 30    |
| HCMV Late Events                                             | 1.533330  | 45      | 12    |
| SUMOylation of DNA replication proteins                      | 1.535361  | 38      | 15    |
| Transport of the SLBP independent Mature mRNA                | 1.536768  | 27      | 7     |
| Citric acid cycle (TCA cycle)                                | 1.545413  | 30      | 15    |
| Postmitotic nuclear pore complex (NPC) reformation           | 1.545697  | 23      | 9     |
| Chromosome Maintenance                                       | 1.546066  | 79      | 33    |
| SARS-CoV-1-host interactions                                 | 1.547713  | 74      | 32    |
| Regulation of PTEN stability and activity                    | 1.551231  | 59      | 22    |
| Late Phase of HIV Life Cycle                                 | 1.551941  | 114     | 46    |
| Nuclear Envelope (NE) Reassembly                             | 1.552482  | 50      | 20    |
| Transport of Ribonucleoproteins into the Host Nucleus        | 1.555139  | 25      | 7     |
| rRNA processing in the nucleus and cytosol                   | 1.557445  | 162     | 58    |
| MAPK6/MAPK4 signaling                                        | 1.560974  | 75      | 27    |
| Base Excision Repair                                         | 1.561452  | 37      | 13    |
| Transport of Mature mRNA derived from an Intron-Containing Transcript | 1.569210  | 60      | 25    |
| Resolution of Sister Chromatid Cohesion                      | 1.569948  | 93      | 33    |
| APC/C-mediated degradation of cell cycle proteins            | 1.577802  | 77      | 32    |
| Regulation of mitotic cell cycle                             | 1.577802  | 77      | 32    |
| Nuclear signaling by ERBB4                                   | 1.580528  | 18      | 6     |
| Vpr-mediated nuclear import of PICs                          | 1.583347  | 25      | 7     |
| RNA Polymerase II Transcription Termination                  | 1.584898  | 55      | 24    |
| Export of Viral Ribonucleoproteins from Nucleus              | 1.586452  | 26      | 7     |
| NEP/NS2 Interacts with the Cellular Export Machinery         | 1.586452  | 26      | 7     |
| RHO GTPase Effectors                                         | 1.587712  | 196     | 71    |
| Cyclin A/B1/B2 associated events during G2/M transition      | 1.590258  | 18      | 13    |
| Resolution of AP sites via the multiple-nucleotide patch replacement pathway | 1.592961  | 21      | 8     |
| MHC class II antigen presentation                            | 1.593630  | 76      | 20    |
| Disorders of transmembrane transporters                      | 1.593917  | 114     | 40    |
| Transport of Mature mRNAs Derived from Intronless Transcripts | 1.595779  | 34      | 11    |
| Abortive elongation of HIV-1 transcript in the absence of Tat | 1.603469  | 22      | 12    |
| Respiratory electron transport                               | 1.604512  | 83      | 41    |
| TCR signaling                                                | 1.605577  | 86      | 33    |
| SUMOylation of SUMOylation proteins                          | 1.612490  | 28      | 8     |
| Transport of Mature Transcript to Cytoplasm                  | 1.613353  | 68      | 28    |
| Signaling by the B Cell Receptor (BCR)                       | 1.615003  | 92      | 29    |
| Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell | 1.617468  | 27      | 8     |
| Regulation of Glucokinase by Glucokinase Regulatory Protein  | 1.618194  | 24      | 7     |
| Defective TPR may confer susceptibility towards thyroid papillary carcinoma (TPC) | 1.618194  | 24      | 7     |
| TP53 regulates transcription of additional cell cycle genes whose exact role in the p53 pathway remain uncertain | 1.620695  | 17      | 9     |
| Cell Cycle, Mitotic                                          | 1.621489  | 403     | 131   |
| Translation                                                  | 1.623650  | 249     | 89    |
| Meiotic synapsis                                             | 1.625306  | 23      | 6     |
| Transcriptional regulation by RUNX1                          | 1.630654  | 142     | 45    |
| Transport of Mature mRNA Derived from an Intronless Transcript | 1.632642  | 33      | 11    |
| Regulation of RAS by GAPs                                    | 1.633814  | 56      | 21    |
| Metabolic disorders of biological oxidation enzymes          | 1.637941  | 16      | 5     |
| Degradation of the extracellular matrix                      | 1.638710  | 63      | 17    |
| Antigen processing-Cross presentation                        | 1.640411  | 75      | 30    |
| M Phase                                                      | 1.641700  | 284     | 97    |
| ER-Phagosome pathway                                         | 1.641933  | 63      | 25    |
| PCNA-Dependent Long Patch Base Excision Repair               | 1.642133  | 17      | 8     |
| SUMOylation of RNA binding proteins                          | 1.647227  | 37      | 13    |
| HIV Life Cycle                                               | 1.657428  | 121     | 51    |
| Recognition of DNA damage by PCNA-containing replication complex | 1.662703  | 25      | 8     |
| Collagen chain trimerization                                 | 1.663230  | 14      | 6     |
| Major pathway of rRNA processing in the nucleolus and cytosol | 1.667105  | 152     | 56    |
| Activation of NF-kappaB in B cells                           | 1.668749  | 57      | 22    |
| Inhibition of DNA recombination at telomere                  | 1.676651  | 21      | 12    |
| SUMOylation of ubiquitinylation proteins                     | 1.677816  | 30      | 9     |
| Assembly of the pre-replicative complex                      | 1.678062  | 76      | 31    |
| Cellular response to starvation                              | 1.680600  | 124     | 53    |
| Regulation of APC/C activators between G1/S and early anaphase | 1.685531  | 71      | 31    |
| RHO GTPases activate IQGAPs                                  | 1.689130  | 12      | 7     |
| Synthesis of DNA                                             | 1.692688  | 106     | 39    |
| RHO GTPases activate PKNs                                    | 1.694726  | 30      | 18    |
| FGFR2 alternative splicing                                   | 1.695097  | 20      | 12    |
| G1/S DNA Damage Checkpoints                                  | 1.701501  | 58      | 23    |
| Lysine catabolism                                            | 1.702366  | 11      | 4     |
| p53-Dependent G1 DNA Damage Response                         | 1.703777  | 57      | 23    |
| p53-Dependent G1/S DNA damage checkpoint                     | 1.703777  | 57      | 23    |
| Synthesis of active ubiquitin: roles of E1 and E2 enzymes    | 1.707925  | 22      | 13    |
| APC/C:Cdc20 mediated degradation of Securin                  | 1.714550  | 58      | 24    |
| Condensation of Prophase Chromosomes                         | 1.715310  | 16      | 8     |
| CDK-mediated phosphorylation and removal of Cdc6             | 1.715389  | 64      | 29    |
| Deposition of new CENPA-containing nucleosomes at the centromere | 1.716501  | 21      | 9     |
| Nucleosome assembly                                          | 1.716501  | 21      | 9     |
| Respiratory electron transport, ATP synthesis by chemiosmotic coupling, and heat production by uncoupling proteins. | 1.717929  | 98      | 49    |
| Downstream signaling events of B Cell Receptor (BCR)         | 1.718758  | 67      | 25    |
| Mitotic G1 phase and G1/S transition                         | 1.726049  | 125     | 44    |
| snRNP Assembly                                               | 1.728076  | 43      | 13    |
| Metabolism of non-coding RNA                                 | 1.728076  | 43      | 13    |
| Hedgehog ligand biogenesis                                   | 1.728994  | 51      | 21    |
| Autodegradation of Cdh1 by Cdh1:APC/C                        | 1.729243  | 56      | 23    |
| S Phase                                                      | 1.730262  | 143     | 49    |
| APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1 | 1.732901  | 64      | 27    |
| Orc1 removal from chromatin                                  | 1.734179  | 64      | 26    |
| SUMOylation of DNA damage response and repair proteins       | 1.736060  | 60      | 19    |
| APC/C:Cdc20 mediated degradation of mitotic proteins         | 1.738294  | 66      | 29    |
| Nuclear events mediated by NFE2L2                            | 1.740403  | 76      | 30    |
| Hh mutants abrogate ligand secretion                         | 1.743268  | 49      | 21    |
| GLI3 is processed to GLI3R by the proteasome                 | 1.743908  | 51      | 21    |
| Regulation of RUNX2 expression and activity                  | 1.750065  | 56      | 21    |
| Interactions of Vpr with host cellular proteins              | 1.752998  | 26      | 8     |
| APC:Cdc20 mediated degradation of cell cycle proteins prior to satisfation of the cell cycle checkpoint | 1.755729  | 65      | 29    |
| Aerobic respiration and respiratory electron transport       | 1.756970  | 160     | 74    |
| Degradation of GLI1 by the proteasome                        | 1.758707  | 50      | 21    |
| Activation of APC/C and APC/C:Cdc20 mediated degradation of mitotic proteins | 1.764399  | 67      | 30    |
| Translesion Synthesis by POLH                                | 1.767317  | 16      | 5     |
| FBXL7 down-regulates AURKA during mitotic entry and in early mitosis | 1.773473  | 48      | 21    |
| tRNA processing in the nucleus                               | 1.775782  | 47      | 12    |
| Dectin-1 mediated noncanonical NF-kB signaling               | 1.780723  | 52      | 21    |
| Degradation of GLI2 by the proteasome                        | 1.790817  | 50      | 21    |
| Detoxification of Reactive Oxygen Species                    | 1.792689  | 22      | 16    |
| Host Interactions of HIV factors                             | 1.800631  | 105     | 40    |
| Metabolism of polyamines                                     | 1.802143  | 53      | 22    |
| Transcriptional regulation by small RNAs                     | 1.806275  | 44      | 20    |
| G1/S Transition                                              | 1.806739  | 113     | 46    |
| Mitotic Anaphase                                             | 1.808689  | 182     | 70    |
| Mitotic Metaphase and Anaphase                               | 1.809050  | 183     | 70    |
| DNA Replication                                              | 1.812655  | 115     | 44    |
| Cdc20:Phospho-APC/C mediated degradation of Cyclin A         | 1.817971  | 64      | 29    |
| Cellular response to hypoxia                                 | 1.823028  | 65      | 26    |
| Cyclin A:Cdk2-associated events at S phase entry             | 1.828468  | 76      | 29    |
| Separation of Sister Chromatids                              | 1.829491  | 147     | 57    |
| NS1 Mediated Effects on Host Pathways                        | 1.830066  | 31      | 10    |
| Viral Messenger RNA Synthesis                                | 1.832078  | 37      | 12    |
| HIV Infection                                                | 1.838232  | 188     | 71    |
| Signaling by NOTCH4                                          | 1.838687  | 66      | 25    |
| DNA Replication Pre-Initiation                               | 1.842178  | 90      | 36    |
| Cyclin E associated events during G1/S transition            | 1.852647  | 74      | 29    |
| Stabilization of p53                                         | 1.856818  | 48      | 20    |
| NIK-->noncanonical NF-kB signaling                           | 1.864619  | 51      | 21    |
| SCF-beta-TrCP mediated degradation of Emi1                   | 1.866171  | 48      | 21    |
| Regulation of mRNA stability by proteins that bind AU-rich elements | 1.870693  | 74      | 32    |
| ABC transporter disorders                                    | 1.870972  | 60      | 26    |
| PCP/CE pathway                                               | 1.872663  | 69      | 27    |
| The role of GTSE1 in G2/M progression after G2 checkpoint    | 1.877632  | 58      | 26    |
| The phototransduction cascade                                | 1.880133  | 11      | 4     |
| Inactivation, recovery and regulation of the phototransduction cascade | 1.880133  | 11      | 4     |
| SARS-CoV-1 modulates host translation machinery              | 1.891711  | 30      | 20    |
| Autodegradation of the E3 ubiquitin ligase COP1              | 1.897373  | 46      | 20    |
| RUNX1 regulates transcription of genes involved in differentiation of HSCs | 1.905666  | 65      | 26    |
| Formation of ATP by chemiosmotic coupling                    | 1.906061  | 12      | 7     |
| Activation of Matrix Metalloproteinases                      | 1.908897  | 11      | 3     |
| Selenoamino acid metabolism                                  | 1.910420  | 95      | 59    |
| Negative regulation of NOTCH4 signaling                      | 1.911847  | 47      | 23    |
| TNFR2 non-canonical NF-kB pathway                            | 1.913089  | 58      | 24    |
| SCF(Skp2)-mediated degradation of p27/p21                    | 1.914750  | 54      | 24    |
| GSK3B and BTRC:CUL1-mediated-degradation of NFE2L2           | 1.920061  | 46      | 21    |
| Vpu mediated degradation of CD4                              | 1.923179  | 45      | 21    |
| Degradation of DVL                                           | 1.942347  | 50      | 22    |
| Asymmetric localization of PCP proteins                      | 1.959794  | 51      | 22    |
| Regulation of Apoptosis                                      | 1.969835  | 47      | 23    |
| Cristae formation                                            | 1.981851  | 24      | 11    |
| Heme biosynthesis                                            | 1.986583  | 11      | 7     |
| ATF6 (ATF6-alpha) activates chaperone genes                  | 1.992543  | 10      | 8     |
| SARS-CoV-2 modulates host translation machinery              | 1.993772  | 39      | 23    |
| Translation initiation complex formation                     | 1.993920  | 50      | 32    |
| Activation of the mRNA upon binding of the cap-binding complex and eIFs, and subsequent binding to 43S | 2.011257  | 51      | 33    |
| Oxygen-dependent proline hydroxylation of Hypoxia-inducible Factor Alpha | 2.014222  | 58      | 24    |
| Nonsense-Mediated Decay (NMD)                                | 2.020740  | 92      | 59    |
| Nonsense Mediated Decay (NMD) enhanced by the Exon Junction Complex (EJC) | 2.020740  | 92      | 59    |
| Cross-presentation of soluble exogenous antigens (endosomes) | 2.021573  | 42      | 20    |
| Ribosomal scanning and start codon recognition               | 2.032561  | 51      | 33    |
| Hh mutants are degraded by ERAD                              | 2.037604  | 48      | 21    |
| Formation of the ternary complex, and subsequently, the 43S complex | 2.037911  | 45      | 21    |
| Regulation of activated PAK-2p34 by proteasome mediated degradation | 2.050765  | 44      | 20    |
| Ubiquitin-dependent degradation of Cyclin D                  | 2.065511  | 46      | 21    |
| Degradation of AXIN                                          | 2.067437  | 48      | 22    |
| Regulation of ornithine decarboxylase (ODC)                  | 2.069842  | 46      | 21    |
| Ubiquitin Mediated Degradation of Phosphorylated Cdc25A      | 2.075990  | 45      | 20    |
| p53-Independent DNA Damage Response                          | 2.075990  | 45      | 20    |
| p53-Independent G1/S DNA damage checkpoint                   | 2.075990  | 45      | 20    |
| Selenocysteine synthesis                                     | 2.096711  | 75      | 51    |
| Mitochondrial protein degradation                            | 2.108916  | 81      | 39    |
| AUF1 (hnRNP D0) binds and destabilizes mRNA                  | 2.115587  | 46      | 21    |
| Eukaryotic Translation Termination                           | 2.123653  | 74      | 52    |
| Collagen degradation                                         | 2.124877  | 30      | 11    |
| Vif-mediated degradation of APOBEC3G                         | 2.128310  | 47      | 22    |
| ATF6 (ATF6-alpha) activates chaperones                       | 2.144804  | 12      | 9     |
| Response of EIF2AK4 (GCN2) to amino acid deficiency          | 2.146668  | 82      | 56    |
| Maternal to zygotic transition (MZT)                         | 2.147872  | 40      | 5     |
| Regulation of RUNX3 expression and activity                  | 2.153411  | 47      | 20    |
| Defective CFTR causes cystic fibrosis                        | 2.153644  | 52      | 24    |
| Influenza Viral RNA Transcription and Replication            | 2.159789  | 110     | 58    |
| Influenza Infection                                          | 2.174912  | 125     | 58    |
| Peptide chain elongation                                     | 2.176142  | 71      | 51    |
| Eukaryotic Translation Elongation                            | 2.197829  | 73      | 52    |
| Metabolism of porphyrins                                     | 2.211398  | 17      | 10    |
| SRP-dependent cotranslational protein targeting to membrane  | 2.220490  | 89      | 61    |
| Signaling by ROBO receptors                                  | 2.257215  | 165     | 76    |
| Viral mRNA Translation                                       | 2.257290  | 70      | 51    |
| Nonsense Mediated Decay (NMD) independent of the Exon Junction Complex (EJC) | 2.298739  | 75      | 53    |
| Eukaryotic Translation Initiation                            | 2.312568  | 98      | 65    |
| Cap-dependent Translation Initiation                         | 2.312568  | 98      | 65    |
| L13a-mediated translational silencing of Ceruloplasmin expression | 2.371951  | 91      | 62    |
| Regulation of expression of SLITs and ROBOs                  | 2.388212  | 135     | 67    |
| Formation of a pool of free 40S subunits                     | 2.389773  | 83      | 59    |
| GTP hydrolysis and joining of the 60S ribosomal subunit      | 2.392154  | 92      | 63    |

### IV-HD

| Description                                                  | NES       | setSize | Count |
| ------------------------------------------------------------ | --------- | ------- | ----- |
| Signaling by CSF1 (M-CSF) in myeloid cells                   | -1.869259 | 25      | 13    |
| PPARA activates gene expression                              | -1.810673 | 75      | 27    |
| Regulation of lipid metabolism by PPARalpha                  | -1.774851 | 77      | 27    |
| Negative regulation of MET activity                          | -1.772842 | 13      | 10    |
| Interleukin-6 family signaling                               | -1.764369 | 11      | 6     |
| Downregulation of SMAD2/3:SMAD4 transcriptional activity     | -1.760074 | 19      | 8     |
| Plasma lipoprotein remodeling                                | -1.753294 | 17      | 8     |
| Signal amplification                                         | -1.749093 | 13      | 6     |
| XBP1(S) activates chaperone genes                            | -1.720464 | 35      | 19    |
| Signaling by EGFR in Cancer                                  | -1.712812 | 14      | 8     |
| FLT3 signaling in disease                                    | -1.705644 | 19      | 12    |
| SUMOylation of DNA methylation proteins                      | -1.696502 | 12      | 5     |
| IRE1alpha activates chaperones                               | -1.688171 | 36      | 19    |
| Signaling by ERBB2                                           | -1.678333 | 25      | 10    |
| Sensory processing of sound by outer hair cells of the cochlea | -1.669455 | 17      | 9     |
| Signaling by FLT3 fusion proteins                            | -1.634968 | 14      | 9     |
| Platelet sensitization by LDL                                | -1.625555 | 13      | 6     |
| Signaling by FGFR3                                           | -1.609945 | 17      | 8     |
| Signaling by FGFR4                                           | -1.609945 | 17      | 8     |
| Negative regulation of the PI3K/AKT network                  | -1.606407 | 45      | 16    |
| Plasma lipoprotein assembly, remodeling, and clearance       | -1.602755 | 37      | 12    |
| RHOBTB2 GTPase cycle                                         | -1.595576 | 17      | 3     |
| Listeria monocytogenes entry into host cells                 | -1.593565 | 11      | 6     |
| PI5P, PP2A and IER3 Regulate PI3K/AKT Signaling              | -1.591690 | 42      | 15    |
| Constitutive Signaling by Ligand-Responsive EGFR Cancer Variants | -1.572422 | 13      | 7     |
| Signaling by Ligand-Responsive EGFR Variants in Cancer       | -1.572422 | 13      | 7     |
| N-glycan trimming in the ER and Calnexin/Calreticulin cycle  | -1.565189 | 28      | 10    |
| Sensory processing of sound by inner hair cells of the cochlea | -1.559412 | 20      | 12    |
| Spry regulation of FGF signaling                             | -1.556017 | 10      | 4     |
| Negative regulation of FGFR1 signaling                       | -1.556017 | 10      | 4     |
| Negative regulation of FGFR3 signaling                       | -1.556017 | 10      | 4     |
| Negative regulation of FGFR4 signaling                       | -1.556017 | 10      | 4     |
| Sensory processing of sound                                  | -1.551170 | 23      | 13    |
| Chondroitin sulfate/dermatan sulfate metabolism              | -1.533951 | 17      | 6     |
| MyD88:MAL(TIRAP) cascade initiated on plasma membrane        | -1.521704 | 56      | 22    |
| Toll Like Receptor TLR1:TLR2 Cascade                         | -1.521704 | 56      | 22    |
| Toll Like Receptor TLR6:TLR2 Cascade                         | -1.521704 | 56      | 22    |
| Toll Like Receptor 2 (TLR2) Cascade                          | -1.521704 | 56      | 22    |
| Constitutive Signaling by Aberrant PI3K in Cancer            | -1.518995 | 24      | 10    |
| Interferon gamma signaling                                   | -1.510473 | 30      | 14    |
| G alpha (12/13) signalling events                            | -1.502975 | 38      | 21    |
| Transcriptional regulation of white adipocyte differentiation | -1.487524 | 55      | 16    |
| Platelet homeostasis                                         | -1.483787 | 30      | 14    |
| GPCR downstream signalling                                   | -1.376518 | 160     | 63    |
| Signaling by GPCR                                            | -1.346686 | 168     | 66    |
| Metabolism of lipids                                         | -1.289286 | 384     | 98    |
| Adaptive Immune System                                       | 1.209229  | 414     | 107   |
| Nervous system development                                   | 1.250303  | 301     | 107   |
| Transcriptional Regulation by TP53                           | 1.259967  | 242     | 88    |
| mRNA Splicing                                                | 1.283339  | 172     | 71    |
| Processing of Capped Intron-Containing Pre-mRNA              | 1.291811  | 226     | 89    |
| Axon guidance                                                | 1.320393  | 292     | 106   |
| mRNA Splicing - Major Pathway                                | 1.332344  | 168     | 70    |
| Signaling by Nuclear Receptors                               | 1.339471  | 130     | 27    |
| Late Phase of HIV Life Cycle                                 | 1.357150  | 108     | 38    |
| Resolution of Sister Chromatid Cohesion                      | 1.360317  | 88      | 35    |
| Signaling by Hedgehog                                        | 1.368366  | 83      | 25    |
| Host Interactions of HIV factors                             | 1.385170  | 101     | 40    |
| Biological oxidations                                        | 1.398811  | 67      | 21    |
| Mitotic Prophase                                             | 1.403700  | 64      | 23    |
| Transcription-Coupled Nucleotide Excision Repair (TC-NER)    | 1.407687  | 61      | 20    |
| Downstream signaling events of B Cell Receptor (BCR)         | 1.427761  | 63      | 18    |
| Cellular response to chemical stress                         | 1.429464  | 145     | 53    |
| Regulation of RUNX2 expression and activity                  | 1.433935  | 53      | 20    |
| Cell Cycle Checkpoints                                       | 1.437531  | 196     | 63    |
| Nuclear Envelope (NE) Reassembly                             | 1.442580  | 49      | 16    |
| HIV Life Cycle                                               | 1.455856  | 114     | 42    |
| Degradation of DVL                                           | 1.464981  | 47      | 22    |
| Mitotic G2-G2/M phases                                       | 1.470910  | 134     | 48    |
| G2/M Transition                                              | 1.470910  | 134     | 48    |
| Regulation of RAS by GAPs                                    | 1.473905  | 53      | 21    |
| Complex I biogenesis                                         | 1.478627  | 42      | 19    |
| Respiratory electron transport                               | 1.485687  | 78      | 30    |
| Diseases of programmed cell death                            | 1.494375  | 35      | 11    |
| Pre-NOTCH Transcription and Translation                      | 1.503348  | 24      | 8     |
| ECM proteoglycans                                            | 1.505102  | 29      | 10    |
| Hedgehog 'off' state                                         | 1.510686  | 68      | 29    |
| Deadenylation-dependent mRNA decay                           | 1.513691  | 40      | 24    |
| Extension of Telomeres                                       | 1.515992  | 39      | 16    |
| Maternal to zygotic transition (MZT)                         | 1.518402  | 34      | 18    |
| Branched-chain amino acid catabolism                         | 1.542656  | 15      | 8     |
| Signaling by NOTCH4                                          | 1.546589  | 60      | 23    |
| Reproduction                                                 | 1.546975  | 42      | 14    |
| Polymerase switching on the C-strand of the telomere         | 1.547057  | 17      | 8     |
| Degradation of GLI2 by the proteasome                        | 1.551086  | 48      | 20    |
| GLI3 is processed to GLI3R by the proteasome                 | 1.551086  | 48      | 20    |
| MAPK6/MAPK4 signaling                                        | 1.557136  | 66      | 27    |
| Chromatin modifications during the maternal to zygotic transition (MZT) | 1.560145  | 10      | 5     |
| G2/M Checkpoints                                             | 1.562135  | 107     | 36    |
| ATF6 (ATF6-alpha) activates chaperones                       | 1.567495  | 12      | 6     |
| Dectin-1 mediated noncanonical NF-kB signaling               | 1.568827  | 50      | 20    |
| HIV Infection                                                | 1.574117  | 179     | 70    |
| Stabilization of p53                                         | 1.574576  | 46      | 19    |
| Condensation of Prometaphase Chromosomes                     | 1.580346  | 10      | 6     |
| Diseases of mitotic cell cycle                               | 1.580636  | 25      | 7     |
| Degradation of GLI1 by the proteasome                        | 1.581502  | 47      | 20    |
| Meiosis                                                      | 1.584745  | 36      | 12    |
| M Phase                                                      | 1.584761  | 260     | 67    |
| Regulation of PTEN stability and activity                    | 1.590671  | 54      | 23    |
| Cell Cycle                                                   | 1.592066  | 445     | 110   |
| Regulation of mRNA stability by proteins that bind AU-rich elements | 1.592218  | 68      | 27    |
| TNFR2 non-canonical NF-kB pathway                            | 1.594814  | 55      | 23    |
| Transcriptional regulation by RUNX1                          | 1.595005  | 134     | 32    |
| APC/C-mediated degradation of cell cycle proteins            | 1.598256  | 72      | 29    |
| Regulation of mitotic cell cycle                             | 1.598256  | 72      | 29    |
| Autodegradation of the E3 ubiquitin ligase COP1              | 1.598353  | 45      | 19    |
| Activation of ATR in response to replication stress          | 1.598869  | 33      | 13    |
| Hedgehog 'on' state                                          | 1.601294  | 54      | 23    |
| Removal of the Flap Intermediate from the C-strand           | 1.602114  | 14      | 7     |
| Neddylation                                                  | 1.602366  | 164     | 46    |
| G0 and Early G1                                              | 1.604597  | 22      | 8     |
| PCP/CE pathway                                               | 1.617148  | 62      | 24    |
| Respiratory electron transport, ATP synthesis by chemiosmotic coupling, and heat production by uncoupling proteins. | 1.619491  | 92      | 37    |
| FBXL7 down-regulates AURKA during mitotic entry and in early mitosis | 1.621595  | 46      | 20    |
| ESR-mediated signaling                                       | 1.622440  | 100     | 25    |
| Recognition of DNA damage by PCNA-containing replication complex | 1.631508  | 24      | 8     |
| SCF-beta-TrCP mediated degradation of Emi1                   | 1.631579  | 47      | 20    |
| Cellular response to hypoxia                                 | 1.631995  | 62      | 25    |
| NFE2L2 regulating anti-oxidant/detoxification enzymes        | 1.632816  | 13      | 8     |
| Lagging Strand Synthesis                                     | 1.632984  | 18      | 9     |
| Mitochondrial iron-sulfur cluster biogenesis                 | 1.635088  | 11      | 7     |
| AUF1 (hnRNP D0) binds and destabilizes mRNA                  | 1.645605  | 45      | 19    |
| RUNX1 regulates transcription of genes involved in differentiation of HSCs | 1.647523  | 63      | 26    |
| Cyclin A/B1/B2 associated events during G2/M transition      | 1.649633  | 17      | 8     |
| NIK-->noncanonical NF-kB signaling                           | 1.650658  | 49      | 20    |
| Orc1 removal from chromatin                                  | 1.653949  | 62      | 24    |
| Telomere C-strand (Lagging Strand) Synthesis                 | 1.654943  | 25      | 12    |
| GSK3B and BTRC:CUL1-mediated-degradation of NFE2L2           | 1.655468  | 45      | 20    |
| DNA Damage Bypass                                            | 1.655633  | 37      | 11    |
| Activation of the pre-replicative complex                    | 1.657165  | 28      | 12    |
| Formation of ATP by chemiosmotic coupling                    | 1.658186  | 12      | 10    |
| Switching of origins to a post-replicative state             | 1.660390  | 76      | 25    |
| MET activates PTK2 signaling                                 | 1.662472  | 12      | 4     |
| Mitotic Anaphase                                             | 1.663883  | 172     | 53    |
| Beta-catenin independent WNT signaling                       | 1.666858  | 84      | 33    |
| Formation of tubulin folding intermediates by CCT/TriC       | 1.668526  | 12      | 7     |
| Cellular response to starvation                              | 1.672384  | 118     | 61    |
| p53-Dependent G1 DNA Damage Response                         | 1.678593  | 51      | 21    |
| p53-Dependent G1/S DNA damage checkpoint                     | 1.678593  | 51      | 21    |
| RHO GTPases activate IQGAPs                                  | 1.681201  | 11      | 3     |
| Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell | 1.682871  | 23      | 5     |
| Mitotic Metaphase and Anaphase                               | 1.683770  | 173     | 54    |
| Resolution of AP sites via the multiple-nucleotide patch replacement pathway | 1.684261  | 20      | 8     |
| Vpu mediated degradation of CD4                              | 1.686267  | 44      | 20    |
| Aberrant regulation of mitotic cell cycle due to RB1 defects | 1.688310  | 23      | 7     |
| Resolution of Abasic Sites (AP sites)                        | 1.690718  | 26      | 11    |
| Telomere Maintenance                                         | 1.691783  | 56      | 23    |
| PCNA-Dependent Long Patch Base Excision Repair               | 1.693297  | 17      | 8     |
| Estrogen-dependent gene expression                           | 1.695618  | 67      | 19    |
| Depolymerization of the Nuclear Lamina                       | 1.697760  | 11      | 6     |
| Chromosome Maintenance                                       | 1.698144  | 72      | 28    |
| M-decay: degradation of maternal mRNAs by maternally stored factors | 1.703932  | 16      | 11    |
| APC:Cdc20 mediated degradation of cell cycle proteins prior to satisfation of the cell cycle checkpoint | 1.705562  | 60      | 25    |
| Transcriptional regulation by small RNAs                     | 1.708220  | 42      | 16    |
| G1/S DNA Damage Checkpoints                                  | 1.708472  | 52      | 22    |
| Citric acid cycle (TCA cycle)                                | 1.709215  | 27      | 14    |
| Hedgehog ligand biogenesis                                   | 1.710239  | 49      | 20    |
| Maturation of TCA enzymes and regulation of TCA cycle        | 1.710956  | 15      | 12    |
| Nuclear events mediated by NFE2L2                            | 1.713082  | 71      | 32    |
| Hh mutants are degraded by ERAD                              | 1.717536  | 47      | 20    |
| Hh mutants abrogate ligand secretion                         | 1.717536  | 47      | 20    |
| Negative regulation of NOTCH4 signaling                      | 1.721080  | 45      | 21    |
| Separation of Sister Chromatids                              | 1.726248  | 138     | 44    |
| Aberrant regulation of mitotic G1/S transition in cancer due to RB1 defects | 1.729533  | 10      | 4     |
| Defective binding of RB1 mutants to E2F1,(E2F2, E2F3)        | 1.729533  | 10      | 4     |
| Meiotic synapsis                                             | 1.730601  | 20      | 4     |
| Cell Cycle, Mitotic                                          | 1.734849  | 368     | 99    |
| Glutathione conjugation                                      | 1.734893  | 13      | 8     |
| Mitotic Telophase/Cytokinesis                                | 1.735760  | 11      | 2     |
| APC/C:Cdc20 mediated degradation of mitotic proteins         | 1.741056  | 61      | 25    |
| Regulation of APC/C activators between G1/S and early anaphase | 1.748150  | 66      | 28    |
| G1/S-Specific Transcription                                  | 1.754016  | 23      | 9     |
| Gastrulation                                                 | 1.757620  | 59      | 25    |
| Translation                                                  | 1.762651  | 230     | 130   |
| rRNA processing                                              | 1.768383  | 164     | 85    |
| Activation of APC/C and APC/C:Cdc20 mediated degradation of mitotic proteins | 1.771163  | 62      | 26    |
| APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1 | 1.775303  | 60      | 25    |
| APC/C:Cdc20 mediated degradation of Securin                  | 1.778978  | 54      | 23    |
| The phototransduction cascade                                | 1.783309  | 11      | 4     |
| Inactivation, recovery and regulation of the phototransduction cascade | 1.783309  | 11      | 4     |
| Aerobic respiration and respiratory electron transport       | 1.784765  | 140     | 57    |
| Transcriptional regulation by RUNX3                          | 1.790810  | 66      | 25    |
| Metabolism of polyamines                                     | 1.794662  | 51      | 25    |
| Regulation of activated PAK-2p34 by proteasome mediated degradation | 1.794734  | 43      | 19    |
| Cdc20:Phospho-APC/C mediated degradation of Cyclin A         | 1.796957  | 59      | 25    |
| Regulation of ornithine decarboxylase (ODC)                  | 1.804391  | 46      | 23    |
| The role of GTSE1 in G2/M progression after G2 checkpoint    | 1.809942  | 56      | 25    |
| SCF(Skp2)-mediated degradation of p27/p21                    | 1.810370  | 51      | 22    |
| Autodegradation of Cdh1 by Cdh1:APC/C                        | 1.817242  | 52      | 23    |
| CDK-mediated phosphorylation and removal of Cdc6             | 1.822288  | 59      | 25    |
| Regulation of Apoptosis                                      | 1.830192  | 44      | 20    |
| TP53 regulates transcription of additional cell cycle genes whose exact role in the p53 pathway remain uncertain | 1.835191  | 15      | 12    |
| Assembly of the pre-replicative complex                      | 1.841773  | 72      | 26    |
| Cristae formation                                            | 1.843428  | 24      | 14    |
| rRNA processing in the nucleus and cytosol                   | 1.862370  | 156     | 83    |
| Mitotic G1 phase and G1/S transition                         | 1.865521  | 116     | 32    |
| Ubiquitin-dependent degradation of Cyclin D                  | 1.867879  | 45      | 20    |
| SARS-CoV-1 modulates host translation machinery              | 1.870299  | 29      | 21    |
| DNA strand elongation                                        | 1.875665  | 29      | 13    |
| Degradation of AXIN                                          | 1.881169  | 47      | 22    |
| Cyclin E associated events during G1/S transition            | 1.883151  | 68      | 29    |
| Condensation of Prophase Chromosomes                         | 1.884843  | 15      | 9     |
| Asymmetric localization of PCP proteins                      | 1.898340  | 46      | 21    |
| Oxygen-dependent proline hydroxylation of Hypoxia-inducible Factor Alpha | 1.902168  | 55      | 25    |
| Cyclin A:Cdk2-associated events at S phase entry             | 1.903061  | 70      | 30    |
| Ubiquitin Mediated Degradation of Phosphorylated Cdc25A      | 1.905882  | 43      | 20    |
| p53-Independent DNA Damage Response                          | 1.905882  | 43      | 20    |
| p53-Independent G1/S DNA damage checkpoint                   | 1.905882  | 43      | 20    |
| Base Excision Repair                                         | 1.917680  | 32      | 14    |
| Major pathway of rRNA processing in the nucleolus and cytosol | 1.926930  | 146     | 78    |
| Influenza Infection                                          | 1.933389  | 122     | 70    |
| Regulation of RUNX3 expression and activity                  | 1.948693  | 46      | 20    |
| Influenza Viral RNA Transcription and Replication            | 1.967828  | 108     | 66    |
| Synthesis of DNA                                             | 1.972000  | 101     | 37    |
| SARS-CoV-2 modulates host translation machinery              | 1.982142  | 37      | 22    |
| Cross-presentation of soluble exogenous antigens (endosomes) | 2.001799  | 41      | 20    |
| Vif-mediated degradation of APOBEC3G                         | 2.007795  | 45      | 22    |
| Nonsense-Mediated Decay (NMD)                                | 2.013469  | 89      | 50    |
| Nonsense Mediated Decay (NMD) enhanced by the Exon Junction Complex (EJC) | 2.013469  | 89      | 50    |
| DNA Replication Pre-Initiation                               | 2.015885  | 86      | 33    |
| SRP-dependent cotranslational protein targeting to membrane  | 2.017591  | 88      | 53    |
| G1/S Transition                                              | 2.026620  | 107     | 42    |
| DNA Replication                                              | 2.039751  | 110     | 43    |
| Somitogenesis                                                | 2.052897  | 40      | 20    |
| Mitochondrial protein degradation                            | 2.057051  | 73      | 37    |
| Detoxification of Reactive Oxygen Species                    | 2.057222  | 20      | 12    |
| Translation initiation complex formation                     | 2.087305  | 49      | 31    |
| Activation of the mRNA upon binding of the cap-binding complex and eIFs, and subsequent binding to 43S | 2.088340  | 50      | 32    |
| Formation of paraxial mesoderm                               | 2.096674  | 46      | 22    |
| Signaling by ROBO receptors                                  | 2.098334  | 156     | 85    |
| Metabolism of amino acids and derivatives                    | 2.111932  | 224     | 117   |
| Formation of the ternary complex, and subsequently, the 43S complex | 2.126980  | 44      | 30    |
| Heme biosynthesis                                            | 2.128247  | 11      | 7     |
| Ribosomal scanning and start codon recognition               | 2.140236  | 50      | 32    |
| S Phase                                                      | 2.163457  | 133     | 46    |
| Selenoamino acid metabolism                                  | 2.168885  | 91      | 57    |
| Selenocysteine synthesis                                     | 2.171936  | 72      | 46    |
| Viral mRNA Translation                                       | 2.178346  | 69      | 46    |
| Nonsense Mediated Decay (NMD) independent of the Exon Junction Complex (EJC) | 2.206450  | 74      | 52    |
| Eukaryotic Translation Termination                           | 2.209620  | 72      | 47    |
| Peptide chain elongation                                     | 2.240268  | 69      | 48    |
| Eukaryotic Translation Elongation                            | 2.250433  | 71      | 47    |
| Metabolism of porphyrins                                     | 2.263622  | 16      | 11    |
| Response of EIF2AK4 (GCN2) to amino acid deficiency          | 2.265917  | 80      | 51    |
| Eukaryotic Translation Initiation                            | 2.325314  | 97      | 64    |
| Cap-dependent Translation Initiation                         | 2.325314  | 97      | 64    |
| L13a-mediated translational silencing of Ceruloplasmin expression | 2.365928  | 90      | 61    |
| Regulation of expression of SLITs and ROBOs                  | 2.373838  | 131     | 76    |
| GTP hydrolysis and joining of the 60S ribosomal subunit      | 2.394554  | 91      | 57    |
| Formation of a pool of free 40S subunits                     | 2.394803  | 82      | 55    |

## 10 WPI

### DNA vaccine

| Description                 | NES       | setSize | Count |
| --------------------------- | --------- | ------- | ----- |
| Striated Muscle Contraction | -2.078391 | 27      | 12    |
| Cell Cycle, Mitotic         | 1.325285  | 426     | 288   |
| RHO GTPase Effectors        | 1.394363  | 214     | 124   |

### EOMES

| Description     | NES      | setSize | Count |
| --------------- | -------- | ------- | ----- |
| Neuronal System | 1.394553 | 283     | 99    |

### GATA3

| Description                                                  | NES       | setSize | Count |
| ------------------------------------------------------------ | --------- | ------- | ----- |
| rRNA modification in the nucleus and cytosol                 | -1.761944 | 57      | 55    |
| Striated Muscle Contraction                                  | -1.626816 | 27      | 9     |
| Signaling by GPCR                                            | 1.191060  | 389     | 198   |
| GPCR downstream signalling                                   | 1.193191  | 348     | 168   |
| SLC-mediated transmembrane transport                         | 1.225929  | 177     | 72    |
| Transmission across Chemical Synapses                        | 1.241173  | 189     | 89    |
| GPCR ligand binding                                          | 1.250982  | 200     | 99    |
| G alpha (i) signalling events                                | 1.259988  | 156     | 81    |
| Neuronal System                                              | 1.287592  | 283     | 142   |
| Class A/1 (Rhodopsin-like receptors)                         | 1.299060  | 139     | 71    |
| Protein-protein interactions at synapses                     | 1.312714  | 65      | 36    |
| Degradation of the extracellular matrix                      | 1.321021  | 86      | 32    |
| Potassium Channels                                           | 1.328675  | 65      | 34    |
| Post-translational protein phosphorylation                   | 1.334892  | 66      | 20    |
| Sensory processing of sound by inner hair cells of the cochlea | 1.353306  | 54      | 26    |
| Sensory processing of sound                                  | 1.355838  | 57      | 27    |
| Visual phototransduction                                     | 1.372641  | 59      | 30    |
| Cell-cell junction organization                              | 1.374837  | 64      | 30    |
| Sensory Perception                                           | 1.394828  | 134     | 65    |
| Adherens junctions interactions                              | 1.405572  | 45      | 20    |

### IV-HD

| Description                 | NES       | setSize | Count |
| --------------------------- | --------- | ------- | ----- |
| Striated Muscle Contraction | -2.643247 | 27      | 15    |

### IV-LD

| Description                                                  | NES       | setSize | Count |
| ------------------------------------------------------------ | --------- | ------- | ----- |
| DNA strand elongation                                        | -2.954534 | 29      | 20    |
| Activation of the pre-replicative complex                    | -2.823981 | 28      | 20    |
| Activation of ATR in response to replication stress          | -2.664441 | 33      | 19    |
| Unwinding of DNA                                             | -2.513751 | 11      | 10    |
| Processive synthesis on the lagging strand                   | -2.438397 | 14      | 10    |
| Lagging Strand Synthesis                                     | -2.432991 | 18      | 10    |
| Removal of the Flap Intermediate                             | -2.412073 | 13      | 10    |
| Telomere C-strand (Lagging Strand) Synthesis                 | -2.368757 | 28      | 13    |
| Polymerase switching on the C-strand of the telomere         | -2.212591 | 20      | 10    |
| Polymerase switching                                         | -2.190069 | 12      | 7     |
| Leading Strand Synthesis                                     | -2.190069 | 12      | 7     |
| tRNA processing                                              | -2.178221 | 88      | 31    |
| tRNA modification in the nucleus and cytosol                 | -2.166711 | 33      | 18    |
| mRNA decay by 3' to 5' exoribonuclease                       | -2.158842 | 14      | 7     |
| Mismatch repair (MMR) directed by MSH2:MSH6 (MutSalpha)      | -2.108978 | 13      | 7     |
| Nuclear import of Rev protein                                | -2.061305 | 29      | 12    |
| Chromosome Maintenance                                       | -2.025494 | 80      | 36    |
| Mitochondrial translation                                    | -2.003138 | 82      | 40    |
| Synthesis of glycosylphosphatidylinositol (GPI)              | -1.990159 | 14      | 7     |
| Interactions of Vpr with host cellular proteins              | -1.976322 | 27      | 11    |
| Extension of Telomeres                                       | -1.967876 | 45      | 23    |
| Mitochondrial translation elongation                         | -1.967231 | 76      | 29    |
| Transport of the SLBP Dependant Mature mRNA                  | -1.957045 | 29      | 12    |
| Mitochondrial translation initiation                         | -1.952516 | 76      | 28    |
| Transcriptional regulation by small RNAs                     | -1.933773 | 45      | 29    |
| Mitochondrial translation termination                        | -1.923750 | 76      | 37    |
| Postmitotic nuclear pore complex (NPC) reformation           | -1.913919 | 24      | 10    |
| tRNA processing in the nucleus                               | -1.905964 | 48      | 17    |
| HDR through Homologous Recombination (HRR)                   | -1.900717 | 56      | 20    |
| Mismatch Repair                                              | -1.894024 | 14      | 7     |
| Diseases of DNA repair                                       | -1.889344 | 43      | 13    |
| Regulation of Glucokinase by Glucokinase Regulatory Protein  | -1.883906 | 25      | 10    |
| Defective TPR may confer susceptibility towards thyroid papillary carcinoma (TPC) | -1.883906 | 25      | 10    |
| Telomere Maintenance                                         | -1.878765 | 62      | 28    |
| Transport of Ribonucleoproteins into the Host Nucleus        | -1.860285 | 26      | 10    |
| RNA Polymerase I Promoter Escape                             | -1.843944 | 30      | 11    |
| Transport of the SLBP independent Mature mRNA                | -1.841741 | 28      | 11    |
| Vpr-mediated nuclear import of PICs                          | -1.825700 | 26      | 10    |
| Transport of Mature mRNAs Derived from Intronless Transcripts | -1.821860 | 35      | 13    |
| snRNP Assembly                                               | -1.819523 | 44      | 19    |
| Metabolism of non-coding RNA                                 | -1.819523 | 44      | 19    |
| SUMOylation of SUMOylation proteins                          | -1.814853 | 29      | 11    |
| DNA Replication Pre-Initiation                               | -1.805919 | 90      | 43    |
| Interactions of Rev with host cellular proteins              | -1.801442 | 31      | 12    |
| TRAF6 mediated IRF7 activation in TLR7/8 or 9 signaling      | -1.781758 | 10      | 4     |
| Rev-mediated nuclear export of HIV RNA                       | -1.722768 | 29      | 11    |
| Deposition of new CENPA-containing nucleosomes at the centromere | -1.716512 | 21      | 9     |
| Nucleosome assembly                                          | -1.716512 | 21      | 9     |
| Transport of Mature mRNA Derived from an Intronless Transcript | -1.710378 | 34      | 12    |
| E2F mediated regulation of DNA replication                   | -1.703348 | 16      | 7     |
| Export of Viral Ribonucleoproteins from Nucleus              | -1.701043 | 27      | 10    |
| NEP/NS2 Interacts with the Cellular Export Machinery         | -1.701043 | 27      | 10    |
| Mismatch repair (MMR) directed by MSH2:MSH3 (MutSbeta)       | -1.691194 | 13      | 6     |
| Resolution of D-loop Structures through Holliday Junction Intermediates | -1.685023 | 30      | 11    |
| Presynaptic phase of homologous DNA pairing and strand exchange | -1.675385 | 32      | 12    |
| Resolution of D-Loop Structures                              | -1.663293 | 31      | 11    |
| Meiotic recombination                                        | -1.660692 | 22      | 8     |
| SUMOylation of ubiquitinylation proteins                     | -1.643981 | 31      | 11    |
| Diseases of DNA Double-Strand Break Repair                   | -1.640567 | 33      | 12    |
| Defective homologous recombination repair (HRR) due to BRCA2 loss of function | -1.640567 | 33      | 12    |
| Homologous DNA Pairing and Strand Exchange                   | -1.628077 | 35      | 12    |
| Nuclear Pore Complex (NPC) Disassembly                       | -1.610885 | 31      | 11    |
| Defective homologous recombination repair (HRR) due to BRCA1 loss of function | -1.605874 | 20      | 8     |
| Defective homologous recombination repair (HRR) due to PALB2 loss of function | -1.605874 | 20      | 8     |
| Defective HDR through Homologous Recombination Repair (HRR) due to PALB2 loss of BRCA1 binding function | -1.605874 | 20      | 8     |
| Defective HDR through Homologous Recombination Repair (HRR) due to PALB2 loss of BRCA2/RAD51/RAD51C binding function | -1.605874 | 20      | 8     |
| Impaired BRCA2 binding to PALB2                              | -1.605874 | 20      | 8     |
| Gap-filling DNA repair synthesis and ligation in GG-NER      | -1.583184 | 20      | 7     |
| DNA Replication                                              | -1.578501 | 115     | 45    |
| Resolution of D-loop Structures through Synthesis-Dependent Strand Annealing (SDSA) | -1.570014 | 22      | 8     |
| RNA Polymerase I Transcription Termination                   | -1.552286 | 26      | 8     |
| Gap-filling DNA repair synthesis and ligation in TC-NER      | -1.539402 | 53      | 29    |
| Dual incision in TC-NER                                      | -1.537704 | 53      | 28    |
| Viral Messenger RNA Synthesis                                | -1.529689 | 38      | 11    |
| Negative epigenetic regulation of rRNA expression            | -1.509520 | 46      | 13    |
| Synthesis of DNA                                             | -1.504299 | 106     | 44    |
| NS1 Mediated Effects on Host Pathways                        | -1.489298 | 32      | 10    |
| Fanconi Anemia Pathway                                       | -1.487228 | 33      | 12    |
| NoRC negatively regulates rRNA expression                    | -1.486856 | 43      | 12    |
| SUMOylation of DNA replication proteins                      | -1.478088 | 39      | 14    |
| RNA Polymerase I Promoter Clearance                          | -1.458619 | 48      | 17    |
| RNA Polymerase I Transcription                               | -1.443085 | 49      | 17    |
| Formation of the ternary complex, and subsequently, the 43S complex | -1.421806 | 45      | 44    |
| Transport of Mature mRNA derived from an Intron-Containing Transcript | -1.408246 | 61      | 18    |
| Transport of Mature Transcript to Cytoplasm                  | -1.403856 | 69      | 20    |
| Assembly of the pre-replicative complex                      | -1.391016 | 76      | 33    |
| Homology Directed Repair                                     | -1.349732 | 89      | 28    |
| HDR through Homologous Recombination (HRR) or Single Strand Annealing (SSA) | -1.325622 | 83      | 23    |
| DNA Double-Strand Break Repair                               | -1.314092 | 114     | 34    |
| Axon guidance                                                | 1.239477  | 361     | 103   |
| Hemostasis                                                   | 1.249844  | 362     | 101   |
| MAPK1/MAPK3 signaling                                        | 1.251507  | 175     | 64    |
| Nervous system development                                   | 1.264132  | 376     | 110   |
| RAF/MAP kinase cascade                                       | 1.265024  | 172     | 63    |
| Extracellular matrix organization                            | 1.266893  | 157     | 59    |
| Signaling by GPCR                                            | 1.267244  | 258     | 111   |
| GPCR downstream signalling                                   | 1.267495  | 241     | 104   |
| RAC1 GTPase cycle                                            | 1.286911  | 148     | 79    |
| RHO GTPase cycle                                             | 1.310042  | 363     | 180   |
| Adipogenesis                                                 | 1.324017  | 81      | 34    |
| Ion channel transport                                        | 1.333062  | 89      | 23    |
| Cargo recognition for clathrin-mediated endocytosis          | 1.351754  | 65      | 18    |
| Factors involved in megakaryocyte development and platelet production | 1.353268  | 98      | 40    |
| Cell-Cell communication                                      | 1.354079  | 87      | 44    |
| G alpha (q) signalling events                                | 1.356045  | 88      | 32    |
| Transport of small molecules                                 | 1.361181  | 417     | 116   |
| CDC42 GTPase cycle                                           | 1.376493  | 125     | 75    |
| SLC-mediated transmembrane transport                         | 1.376789  | 129     | 35    |
| Transcriptional regulation of white adipocyte differentiation | 1.387855  | 62      | 25    |
| Cell surface interactions at the vascular wall               | 1.396451  | 77      | 39    |
| Fatty acid metabolism                                        | 1.396874  | 119     | 40    |
| Degradation of the extracellular matrix                      | 1.404099  | 67      | 27    |
| Peptide ligand-binding receptors                             | 1.413956  | 37      | 15    |
| RHOD GTPase cycle                                            | 1.422377  | 45      | 28    |
| NRAGE signals death through JNK                              | 1.423426  | 43      | 20    |
| RHOJ GTPase cycle                                            | 1.425257  | 48      | 31    |
| L1CAM interactions                                           | 1.431323  | 67      | 32    |
| Circadian Clock                                              | 1.436316  | 56      | 23    |
| RND3 GTPase cycle                                            | 1.436619  | 36      | 14    |
| TBC/RABGAPs                                                  | 1.439978  | 34      | 17    |
| Signaling by NOTCH3                                          | 1.449675  | 33      | 9     |
| RHOA GTPase cycle                                            | 1.451754  | 127     | 53    |
| Cardiac conduction                                           | 1.459053  | 49      | 15    |
| Arachidonic acid metabolism                                  | 1.462290  | 31      | 5     |
| Metabolism of water-soluble vitamins and cofactors           | 1.466094  | 88      | 27    |
| Sensory processing of sound by outer hair cells of the cochlea | 1.478480  | 25      | 9     |
| Signaling by ERBB2 in Cancer                                 | 1.481454  | 17      | 8     |
| Signaling by ERBB2 KD Mutants                                | 1.481454  | 17      | 8     |
| Uptake and actions of bacterial toxins                       | 1.484117  | 16      | 6     |
| Nucleotide catabolism                                        | 1.486791  | 24      | 4     |
| EGFR downregulation                                          | 1.490518  | 21      | 7     |
| Constitutive Signaling by Aberrant PI3K in Cancer            | 1.491620  | 35      | 17    |
| Regulation of lipid metabolism by PPARalpha                  | 1.492182  | 93      | 40    |
| Ion transport by P-type ATPases                              | 1.494421  | 32      | 13    |
| Sensory Perception                                           | 1.494804  | 80      | 33    |
| Glycogen synthesis                                           | 1.494930  | 10      | 2     |
| PPARA activates gene expression                              | 1.497057  | 91      | 40    |
| Bile acid and bile salt metabolism                           | 1.497313  | 22      | 10    |
| Signal transduction by L1                                    | 1.497539  | 20      | 3     |
| Endogenous sterols                                           | 1.500532  | 13      | 7     |
| Assembly of active LPL and LIPC lipase complexes             | 1.504366  | 11      | 5     |
| Glyoxylate metabolism and glycine degradation                | 1.504830  | 14      | 8     |
| Non-integrin membrane-ECM interactions                       | 1.508077  | 35      | 26    |
| Syndecan interactions                                        | 1.509090  | 21      | 13    |
| Biological oxidations                                        | 1.511144  | 94      | 27    |
| Interaction between L1 and Ankyrins                          | 1.514887  | 11      | 6     |
| Attachment and Entry                                         | 1.524825  | 11      | 8     |
| Basigin interactions                                         | 1.525764  | 20      | 7     |
| SUMOylation of intracellular receptors                       | 1.530752  | 19      | 10    |
| Sulfur amino acid metabolism                                 | 1.533032  | 24      | 10    |
| Glycosphingolipid biosynthesis                               | 1.533504  | 10      | 4     |
| NR1H2 and NR1H3-mediated signaling                           | 1.546182  | 32      | 13    |
| Nitric oxide stimulates guanylate cyclase                    | 1.564377  | 10      | 6     |
| Degradation of cysteine and homocysteine                     | 1.564391  | 13      | 6     |
| Signaling by ERBB2 TMD/JMD mutants                           | 1.568626  | 14      | 8     |
| Plasma lipoprotein clearance                                 | 1.569191  | 26      | 11    |
| Ion homeostasis                                              | 1.569326  | 27      | 10    |
| Metabolism of vitamins and cofactors                         | 1.576338  | 131     | 38    |
| CD209 (DC-SIGN) signaling                                    | 1.588426  | 17      | 9     |
| Erythropoietin activates RAS                                 | 1.590764  | 12      | 8     |
| Metabolism of folate and pterines                            | 1.597534  | 11      | 5     |
| Visual phototransduction                                     | 1.606751  | 37      | 20    |
| Phase I - Functionalization of compounds                     | 1.608231  | 41      | 13    |
| Keratinization                                               | 1.608706  | 21      | 11    |
| Formation of the cornified envelope                          | 1.608706  | 21      | 11    |
| Serotonin Neurotransmitter Release Cycle                     | 1.610092  | 10      | 2     |
| SHC1 events in ERBB2 signaling                               | 1.617309  | 13      | 10    |
| NOTCH3 Activation and Transmission of Signal to the Nucleus  | 1.625611  | 18      | 5     |
| Activation of IRF3, IRF7 mediated by TBK1, IKK (IKBKE)      | 1.626696  | 11      | 1     |
| Nuclear Receptor transcription pathway                       | 1.628092  | 32      | 17    |
| Post-translational protein phosphorylation                   | 1.634458  | 51      | 14    |
| BMAL1:CLOCK,NPAS2 activates circadian gene expression        | 1.640093  | 22      | 11    |
| Gluconeogenesis                                              | 1.641454  | 14      | 7     |
| Activation of Matrix Metalloproteinases                      | 1.648243  | 11      | 4     |
| Triglyceride metabolism                                      | 1.649731  | 20      | 10    |
| Cellular hexose transport                                    | 1.662707  | 11      | 6     |
| Metabolism of fat-soluble vitamins                           | 1.665754  | 25      | 13    |
| Triglyceride catabolism                                      | 1.666912  | 13      | 6     |
| Cytochrome P450 - arranged by substrate type                 | 1.671828  | 22      | 7     |
| Regulation of Complement cascade                             | 1.673959  | 11      | 3     |
| Plasma lipoprotein assembly, remodeling, and clearance       | 1.681026  | 46      | 22    |
| Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs) | 1.690045  | 57      | 17    |
| Scavenging by Class A Receptors                              | 1.691384  | 15      | 7     |
| Binding and Uptake of Ligands by Scavenger Receptors         | 1.704992  | 24      | 8     |
| Retinoid metabolism and transport                            | 1.709875  | 21      | 8     |
| Metabolism of steroids                                       | 1.721594  | 97      | 36    |
| Plasma lipoprotein remodeling                                | 1.736262  | 19      | 9     |
| Regulation of cholesterol biosynthesis by SREBP (SREBF)      | 1.743377  | 48      | 21    |
| Activation of gene expression by SREBF (SREBP)               | 1.821012  | 37      | 20    |
| Cholesterol biosynthesis                                     | 1.868195  | 22      | 14    |
| Complement cascade                                           | 1.892080  | 17      | 6     |
| Initial triggering of complement                             | 1.915240  | 11      | 6     |

# Interpretation of GSEA Results

From ChatGPT when prompted to explain a negative *Normalised Enrichment Score* (NES):

*In summary, a negative NES in GSEA indicates that the gene set is significantly downregulated in the condition of interest compared to the control, reflecting a negative correlation with the phenotype being studied. This information can provide valuable insights into the biological processes that are less active or suppressed in the experimental condition.*

- 

  

  Positive NES:

  - 

    Indicates that the gene set is enriched at the top of the ranked list, suggesting that genes in the set are upregulated in the condition of interest.

- 

  

  Negative NES:

  - 

    Indicates that the gene set is enriched at the bottom of the ranked list, suggesting that genes in the set are downregulated in the condition of interest.

Most studies related to viral infection use qPCR to 'measure' gene expression of target genes in relation to a *housekeeping gene*. The approach of using RNAseq as a tool to study immune responses is inherently different, in the sense that it does not compare gene expression against a housekeeping gene. Instead, it requires using a *reference*, such as a control group. This control group provides the baseline gene expression against which all other groups are compared to.

Because with RNAseq one looks at thousands of genes simultaneously, it makes more sense to have a holistic view of gene expression. Any emerging patterns in gene expression tend to stand out during pathway analysis, since pathways, by definition, incorporate hundreds, or even thousands, of differentially regulated genes.

WHAT GENES ARE EXPECTED TO BE DIFFERENTIALLY REGULATED BY EOMES, GATA3, AND DNA VACCINE?

CREATE A TIMELINE PLOT OF GENE EXPRESSION, I.E. 1WPC VS 10WPI, 4WPC VS 1WPC, 6WPC VS 4WPC, 10WPC VS 6WPC FOR CONTROL

Immune-related genes

Housekeeping: -actin (ACTB): AF012125, ENSSSAG00000116649. A lot of reads everywhere.

SasaIFN-: AY216594, no Ensembl

x: U66475, ENSSSAG00070007970 (mxa). No hits in read files.

ISG15: AY926456, ENSSSAG00000105254 (UBIL). Mapped but mostly no reads.

IFN-: AY795563, ENSSSAG00000105299. Mapped but no reads.

F- (1&2): NM_001123589 & NM_001123590, ENSSSAG00000065312 (mapped but no reads) & ENSSSAG00000053783 (no hits)

IL12-: BT049114, ENSSSAG00000009655. Mapped but with few reads.

IL-10: EF165029, ENSSSAG00000107544. Mapped but with no reads.

IL-8: NM_001140710, ENSSSAG00000006498. Mapped but with no reads.

CD3-: NM_001123622, ENSSSAG00000076824. A lot of reads in s and hk. Fewer reads in h.

CD4: NM_001124539, rainbow trout gene

CD8-: NM_001123583, ENSSSAG00000065860. Mapped but with few reads.

TCR-: BT050114, no Ensembl. ENSSSAG00070030723 for T-cell receptor alpha/delta variable 31.0. No hits.

MHC I: AF508864, no Ensembl. ENSSSAG00000077407 for a novel gene. A lot of hits in h, hk, and s.

MHC II: BT049430, no Ensembl. ENSSSAG00000004635 for a novel gene. A LOT of hits everywhere.

How did you look at specific genes in the RNAseq data (genes in Table 1, for example)? Are these manually curated, and you get their respective raw counts from gene count files?