# GSEA research notes - SPLEEN

## 10WPI

### DNA vaccine

ReactomePA differentially regulated pathways.

| Description                                                  | NES       | setSize | Count |
| ------------------------------------------------------------ | --------- | ------- | ----- |
| Regulation of Complement cascade                             | -2.241270 | 15      | 11    |
| Amyloid fiber formation                                      | -2.117998 | 27      | 5     |
| Complement cascade                                           | -1.984553 | 19      | 11    |
| Striated Muscle Contraction                                  | -1.884199 | 14      | 5     |
| Neurotransmitter release cycle                               | -1.783909 | 14      | 6     |
| Peroxisomal protein import                                   | -1.779507 | 27      | 11    |
| GPCR ligand binding                                          | -1.777843 | 68      | 14    |
| Post-translational modification: synthesis of GPI-anchored proteins | -1.766703 | 17      | 12    |
| Metabolic disorders of biological oxidation enzymes          | -1.735140 | 10      | 6     |
| Insulin processing                                           | -1.633392 | 15      | 3     |
| Diseases of metabolism                                       | -1.561942 | 94      | 35    |
| Biosynthesis of the N-glycan precursor (dolichol lipid-linked oligosaccharide, LLO) and transfer to a nascent protein | -1.507741 | 36      | 18    |
| Peptide hormone metabolism                                   | -1.503343 | 31      | 6     |
| Platelet degranulation                                       | -1.494170 | 72      | 14    |
| Class A/1 (Rhodopsin-like receptors)                         | -1.486493 | 51      | 10    |
| Biological oxidations                                        | -1.477513 | 62      | 23    |
| Response to elevated platelet cytosolic Ca2+                 | -1.466929 | 76      | 17    |
| Axon guidance                                                | 1.359716  | 303     | 124   |
| Nervous system development                                   | 1.370982  | 309     | 58    |
| Mitotic Metaphase and Anaphase                               | 1.400823  | 160     | 73    |
| Mitotic Anaphase                                             | 1.400823  | 160     | 73    |
| Cellular Senescence                                          | 1.413021  | 86      | 34    |
| Mitotic G2-G2/M phases                                       | 1.426075  | 121     | 59    |
| G2/M Transition                                              | 1.426075  | 121     | 59    |
| Cell Cycle Checkpoints                                       | 1.428235  | 176     | 77    |
| Cytokine Signaling in Immune system                          | 1.438643  | 359     | 113   |
| G1/S Transition                                              | 1.448713  | 102     | 52    |
| Death Receptor Signaling                                     | 1.457112  | 84      | 26    |
| Mitotic Prometaphase                                         | 1.479377  | 120     | 48    |
| Signaling by NTRK1 (TRKA)                                    | 1.481876  | 65      | 31    |
| TCF dependent signaling in response to WNT                   | 1.485156  | 115     | 33    |
| Ub-specific processing proteases                             | 1.487941  | 115     | 59    |
| Signaling by VEGF                                            | 1.488078  | 77      | 20    |
| Adipogenesis                                                 | 1.504584  | 66      | 24    |
| SARS-CoV-2 activates/modulates innate and adaptive immune responses | 1.506024  | 65      | 22    |
| Signaling by the B Cell Receptor (BCR)                       | 1.509223  | 87      | 47    |
| SARS-CoV-2-host interactions                                 | 1.511130  | 121     | 32    |
| Apoptosis                                                    | 1.513401  | 112     | 53    |
| Activation of anterior HOX genes in hindbrain development during early embryogenesis | 1.513965  | 43      | 16    |
| Activation of HOX genes during differentiation               | 1.513965  | 43      | 16    |
| Interferon Signaling                                         | 1.517474  | 126     | 40    |
| Mitotic Prophase                                             | 1.517506  | 61      | 21    |
| Hedgehog 'off' state                                         | 1.519717  | 69      | 38    |
| Fc epsilon receptor (FCERI) signaling                        | 1.524962  | 99      | 50    |
| RHO GTPase Effectors                                         | 1.526557  | 166     | 73    |
| Signaling by ALK in cancer                                   | 1.528765  | 64      | 21    |
| Signaling by ALK fusions and activated point mutants         | 1.528765  | 64      | 21    |
| CLEC7A (Dectin-1) signaling                                  | 1.529449  | 75      | 40    |
| MAPK1/MAPK3 signaling                                        | 1.529760  | 143     | 39    |
| S Phase                                                      | 1.532588  | 126     | 66    |
| Costimulation by the CD28 family                             | 1.533874  | 39      | 15    |
| G alpha (12/13) signalling events                            | 1.533885  | 38      | 17    |
| Respiratory Syncytial Virus Infection Pathway                | 1.537826  | 67      | 25    |
| RAC1 GTPase cycle                                            | 1.539850  | 126     | 52    |
| RAF/MAP kinase cascade                                       | 1.540032  | 140     | 39    |
| Estrogen-dependent gene expression                           | 1.541615  | 67      | 28    |
| M Phase                                                      | 1.542473  | 234     | 104   |
| Signaling by Nuclear Receptors                               | 1.543015  | 126     | 48    |
| Anchoring of the basal body to the plasma membrane           | 1.546421  | 48      | 19    |
| Signaling by NOTCH1                                          | 1.548265  | 43      | 16    |
| MAPK family signaling cascades                               | 1.549645  | 167     | 79    |
| Regulation of MITF-M-dependent genes involved in apoptosis   | 1.552282  | 11      | 7     |
| Adherens junctions interactions                              | 1.553615  | 25      | 9     |
| Cell Cycle, Mitotic                                          | 1.555357  | 330     | 111   |
| Deubiquitination                                             | 1.558352  | 161     | 50    |
| RHOJ GTPase cycle                                            | 1.564518  | 40      | 19    |
| Cell Cycle                                                   | 1.566349  | 398     | 135   |
| C-type lectin receptors (CLRs)                               | 1.566550  | 86      | 45    |
| RAC3 GTPase cycle                                            | 1.568830  | 61      | 25    |
| Constitutive Signaling by EGFRvIII                           | 1.569113  | 11      | 4     |
| Signaling by EGFRvIII in Cancer                              | 1.569113  | 11      | 4     |
| Transcriptional regulation by RUNX3                          | 1.569290  | 67      | 35    |
| ESR-mediated signaling                                       | 1.573844  | 99      | 38    |
| Signaling by WNT                                             | 1.574110  | 166     | 82    |
| VEGFA-VEGFR2 Pathway                                         | 1.575299  | 70      | 19    |
| Constitutive Signaling by Ligand-Responsive EGFR Cancer Variants | 1.576634  | 13      | 4     |
| Signaling by EGFR in Cancer                                  | 1.576634  | 13      | 4     |
| Signaling by Ligand-Responsive EGFR Variants in Cancer       | 1.576634  | 13      | 4     |
| Interferon gamma signaling                                   | 1.577620  | 35      | 17    |
| Oxidative Stress Induced Senescence                          | 1.579621  | 44      | 21    |
| RIPK1-mediated regulated necrosis                            | 1.582296  | 21      | 6     |
| Regulation of necroptotic cell death                         | 1.582296  | 21      | 6     |
| SARS-CoV-2 Infection                                         | 1.583791  | 178     | 50    |
| Degradation of beta-catenin by the destruction complex       | 1.598609  | 65      | 38    |
| Interferon alpha/beta signaling                              | 1.606583  | 30      | 13    |
| Beta-catenin independent WNT signaling                       | 1.607205  | 90      | 50    |
| SARS-CoV-1 targets host intracellular signalling and regulatory pathways | 1.610306  | 11      | 7     |
| RHO GTPase cycle                                             | 1.612038  | 291     | 115   |
| Transcriptional regulation by RUNX1                          | 1.615748  | 133     | 40    |
| Platelet calcium homeostasis                                 | 1.621076  | 12      | 5     |
| Diseases of signal transduction by growth factor receptors and second messengers | 1.625372  | 284     | 98    |
| Potential therapeutics for SARS                              | 1.626597  | 68      | 35    |
| SARS-CoV Infections                                          | 1.632015  | 259     | 81    |
| Signaling by Rho GTPases, Miro GTPases and RHOBTB3           | 1.634491  | 422     | 164   |
| Programmed Cell Death                                        | 1.635565  | 132     | 51    |
| RHOC GTPase cycle                                            | 1.636843  | 55      | 26    |
| Opioid Signalling                                            | 1.637278  | 38      | 20    |
| NOTCH1 Intracellular Domain Regulates Transcription          | 1.638939  | 29      | 12    |
| RHOF GTPase cycle                                            | 1.641598  | 30      | 16    |
| Signaling by Rho GTPases                                     | 1.645400  | 413     | 161   |
| Condensation of Prophase Chromosomes                         | 1.649244  | 14      | 7     |
| RND2 GTPase cycle                                            | 1.659249  | 31      | 17    |
| Maternal to zygotic transition (MZT)                         | 1.660534  | 33      | 18    |
| RAF activation                                               | 1.662111  | 26      | 12    |
| HSF1-dependent transactivation                               | 1.663105  | 15      | 8     |
| RHOBTB GTPase Cycle                                          | 1.663225  | 25      | 9     |
| RAC2 GTPase cycle                                            | 1.667253  | 66      | 33    |
| Regulation of RUNX1 Expression and Activity                  | 1.669452  | 13      | 9     |
| HDMs demethylate histones                                    | 1.672443  | 16      | 9     |
| RHOA GTPase cycle                                            | 1.673734  | 101     | 47    |
| Sema4D induced cell migration and growth-cone collapse       | 1.681423  | 13      | 6     |
| Mitotic Telophase/Cytokinesis                                | 1.683839  | 10      | 7     |
| Regulation of MECP2 expression and activity                  | 1.687468  | 20      | 12    |
| Signalling to ERKs                                           | 1.688122  | 22      | 13    |
| RND1 GTPase cycle                                            | 1.688583  | 28      | 15    |
| Evasion by RSV of host interferon responses                  | 1.695392  | 13      | 5     |
| Ca2+ pathway                                                 | 1.696858  | 26      | 16    |
| Initiation of Nuclear Envelope (NE) Reformation              | 1.707072  | 13      | 7     |
| Interleukin-3, Interleukin-5 and GM-CSF signaling            | 1.710003  | 31      | 8     |
| RUNX1 regulates genes involved in megakaryocyte differentiation and platelet function | 1.711621  | 32      | 14    |
| RSV-host interactions                                        | 1.712072  | 51      | 23    |
| CDC42 GTPase cycle                                           | 1.712230  | 102     | 44    |
| Trafficking of AMPA receptors                                | 1.714581  | 15      | 5     |
| Glutamate binding, activation of AMPA receptors and synaptic plasticity | 1.714581  | 15      | 5     |
| Translation of Structural Proteins                           | 1.717780  | 18      | 8     |
| RHOB GTPase cycle                                            | 1.718858  | 49      | 26    |
| Chromatin modifying enzymes                                  | 1.734738  | 152     | 58    |
| Chromatin organization                                       | 1.734738  | 152     | 58    |
| Signaling by WNT in cancer                                   | 1.737845  | 17      | 11    |
| Apoptotic cleavage of cellular proteins                      | 1.751430  | 23      | 14    |
| DDX58/IFIH1-mediated induction of interferon-alpha/beta      | 1.754826  | 39      | 11    |
| RHO GTPases activate PAKs                                    | 1.755418  | 15      | 6     |
| Maturation of nucleoprotein                                  | 1.755831  | 13      | 5     |
| Effects of PIP2 hydrolysis                                   | 1.757041  | 16      | 5     |
| Apoptotic execution phase                                    | 1.758126  | 30      | 15    |
| Negative regulators of DDX58/IFIH1 signaling                 | 1.759331  | 20      | 6     |
| Platelet homeostasis                                         | 1.762752  | 32      | 19    |
| RHOBTB2 GTPase cycle                                         | 1.768215  | 16      | 7     |
| E2F mediated regulation of DNA replication                   | 1.774100  | 16      | 8     |
| Meiotic synapsis                                             | 1.776616  | 18      | 9     |
| RHO GTPases activate CIT                                     | 1.801145  | 13      | 6     |
| SARS-CoV-1 Infection                                         | 1.802731  | 95      | 27    |
| NR1H2 and NR1H3-mediated signaling                           | 1.813403  | 22      | 14    |
| RND3 GTPase cycle                                            | 1.814530  | 31      | 18    |
| RHOD GTPase cycle                                            | 1.822609  | 39      | 21    |
| Regulation of signaling by CBL                               | 1.829728  | 19      | 7     |
| Beta-catenin phosphorylation cascade                         | 1.843554  | 11      | 8     |
| Signaling by AXIN mutants                                    | 1.843554  | 11      | 8     |
| Signaling by CTNNB1 phospho-site mutants                     | 1.843554  | 11      | 8     |
| Signaling by APC mutants                                     | 1.843554  | 11      | 8     |
| Signaling by AMER1 mutants                                   | 1.843554  | 11      | 8     |
| Signaling by GSK3beta mutants                                | 1.843554  | 11      | 8     |
| CTNNB1 S33 mutants aren't phosphorylated                     | 1.843554  | 11      | 8     |
| CTNNB1 S37 mutants aren't phosphorylated                     | 1.843554  | 11      | 8     |
| CTNNB1 S45 mutants aren't phosphorylated                     | 1.843554  | 11      | 8     |
| CTNNB1 T41 mutants aren't phosphorylated                     | 1.843554  | 11      | 8     |
| APC truncation mutants have impaired AXIN binding            | 1.843554  | 11      | 8     |
| AXIN missense mutants destabilize the destruction complex    | 1.843554  | 11      | 8     |
| Truncations of AMER1 destabilize the destruction complex     | 1.843554  | 11      | 8     |
| PKMTs methylate histone lysines                              | 1.847820  | 30      | 15    |
| NR1H3 & NR1H2 regulate gene expression linked to cholesterol transport and efflux | 1.848806  | 20      | 13    |
| Disassembly of the destruction complex and recruitment of AXIN to the membrane | 1.872158  | 17      | 11    |
| Negative regulation of MAPK pathway                          | 1.885807  | 26      | 13    |
| TRAF6 mediated IRF7 activation                               | 1.982506  | 11      | 6     |

### EOMES

| Description                                                  | NES       | setSize | Count |
| ------------------------------------------------------------ | --------- | ------- | ----- |
| Signaling by ALK                                             | 1.649581  | 26      | 10    |
| Voltage gated Potassium channels                             | 1.566912  | 29      | 14    |
| TNFR1-induced NF-kappa-B signaling pathway                   | 1.552213  | 28      | 13    |
| Sensory processing of sound                                  | 1.545197  | 57      | 29    |
| Signaling by ERBB4                                           | 1.533551  | 47      | 22    |
| Sensory processing of sound by inner hair cells of the cochlea | 1.530252  | 53      | 26    |
| Neurexins and neuroligins                                    | 1.515727  | 47      | 15    |
| Neuronal System                                              | 1.427153  | 299     | 104   |
| Transmission across Chemical Synapses                        | 1.360267  | 195     | 74    |
| Signaling by Receptor Tyrosine Kinases                       | 1.215053  | 432     | 192   |
| Response to elevated platelet cytosolic Ca2+                 | -1.818670 | 100     | 14    |
| Platelet degranulation                                       | -1.853988 | 96      | 14    |
| Post-translational protein phosphorylation                   | -1.883851 | 76      | 13    |
| Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs) | -1.922559 | 86      | 15    |
| Synthesis of glycosylphosphatidylinositol (GPI)              | -1.976571 | 16      | 7     |
| Initial triggering of complement                             | -1.987510 | 13      | 5     |
| Pyrimidine catabolism                                        | -2.036910 | 11      | 5     |
| Diseases of hemostasis                                       | -2.163460 | 12      | 6     |
| Common Pathway of Fibrin Clot Formation                      | -2.331327 | 11      | 7     |
| Intrinsic Pathway of Fibrin Clot Formation                   | -2.350957 | 13      | 8     |
| Formation of Fibrin Clot (Clotting Cascade)                  | -2.661372 | 22      | 10    |
| Regulation of Complement cascade                             | -2.773996 | 24      | 13    |
| Complement cascade                                           | -2.832269 | 31      | 14    |

### GATA3

| Description                                                  | NES       | setSize | Count |
| ------------------------------------------------------------ | --------- | ------- | ----- |
| CD209 (DC-SIGN) signaling                                    | 1.695850  | 17      | 6     |
| Signaling by Hippo                                           | 1.656248  | 16      | 10    |
| ECM proteoglycans                                            | 1.614106  | 47      | 26    |
| MITF-M-dependent gene expression                             | 1.606173  | 64      | 31    |
| RHOH GTPase cycle                                            | 1.595730  | 30      | 13    |
| MITF-M-regulated melanocyte development                      | 1.533134  | 88      | 38    |
| RHOB GTPase cycle                                            | 1.532649  | 61      | 35    |
| L1CAM interactions                                           | 1.510738  | 68      | 35    |
| RHOA GTPase cycle                                            | 1.457084  | 125     | 60    |
| RHO GTPase cycle                                             | 1.441012  | 363     | 155   |
| Chromatin modifying enzymes                                  | 1.433822  | 175     | 75    |
| Chromatin organization                                       | 1.433822  | 175     | 75    |
| RAC1 GTPase cycle                                            | 1.418491  | 150     | 68    |
| Nervous system development                                   | 1.393143  | 393     | 132   |
| Axon guidance                                                | 1.375631  | 377     | 123   |
| Signaling by Receptor Tyrosine Kinases                       | 1.355121  | 374     | 164   |
| Metabolism of nucleotides                                    | -1.619210 | 76      | 19    |
| Diseases of carbohydrate metabolism                          | -2.005272 | 26      | 5     |
| Post-translational protein phosphorylation                   | -2.012732 | 60      | 10    |
| Cholesterol biosynthesis                                     | -2.041372 | 24      | 9     |
| Tryptophan catabolism                                        | -2.055674 | 10      | 3     |
| Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs) | -2.057657 | 69      | 13    |
| Response to elevated platelet cytosolic Ca2+                 | -2.143473 | 90      | 14    |
| Platelet degranulation                                       | -2.204138 | 86      | 14    |
| Intrinsic Pathway of Fibrin Clot Formation                   | -2.463928 | 12      | 6     |
| Common Pathway of Fibrin Clot Formation                      | -2.465193 | 10      | 8     |
| Regulation of Complement cascade                             | -2.562415 | 21      | 9     |
| Complement cascade                                           | -2.603283 | 26      | 9     |
| Formation of Fibrin Clot (Clotting Cascade)                  | -2.750159 | 21      | 11    |

### IV-HD

nMDS plots? Can I use them?

| Description                                                  | NES       | setSize | Count |
| ------------------------------------------------------------ | --------- | ------- | ----- |
| CD209 (DC-SIGN) signaling                                    | 1.695850  | 17      | 6     |
| Signaling by Hippo                                           | 1.656248  | 16      | 10    |
| ECM proteoglycans                                            | 1.614106  | 47      | 26    |
| MITF-M-dependent gene expression                             | 1.606173  | 64      | 31    |
| RHOH GTPase cycle                                            | 1.595730  | 30      | 13    |
| MITF-M-regulated melanocyte development                      | 1.533134  | 88      | 38    |
| RHOB GTPase cycle                                            | 1.532649  | 61      | 35    |
| L1CAM interactions                                           | 1.510738  | 68      | 35    |
| RHOA GTPase cycle                                            | 1.457084  | 125     | 60    |
| RHO GTPase cycle                                             | 1.441012  | 363     | 155   |
| Chromatin modifying enzymes                                  | 1.433822  | 175     | 75    |
| Chromatin organization                                       | 1.433822  | 175     | 75    |
| RAC1 GTPase cycle                                            | 1.418491  | 150     | 68    |
| Nervous system development                                   | 1.393143  | 393     | 132   |
| Axon guidance                                                | 1.375631  | 377     | 123   |
| Signaling by Receptor Tyrosine Kinases                       | 1.355121  | 374     | 164   |
| Metabolism of nucleotides                                    | -1.619210 | 76      | 19    |
| Diseases of carbohydrate metabolism                          | -2.005272 | 26      | 5     |
| Post-translational protein phosphorylation                   | -2.012732 | 60      | 10    |
| Cholesterol biosynthesis                                     | -2.041372 | 24      | 9     |
| Tryptophan catabolism                                        | -2.055674 | 10      | 3     |
| Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs) | -2.057657 | 69      | 13    |
| Response to elevated platelet cytosolic Ca2+                 | -2.143473 | 90      | 14    |
| Platelet degranulation                                       | -2.204138 | 86      | 14    |
| Intrinsic Pathway of Fibrin Clot Formation                   | -2.463928 | 12      | 6     |
| Common Pathway of Fibrin Clot Formation                      | -2.465193 | 10      | 8     |
| Regulation of Complement cascade                             | -2.562415 | 21      | 9     |
| Complement cascade                                           | -2.603283 | 26      | 9     |
| Formation of Fibrin Clot (Clotting Cascade)                  | -2.750159 | 21      | 11    |

### IV-LD

| Description                                        | NES       | setSize | Count |
| -------------------------------------------------- | --------- | ------- | ----- |
| Striated Muscle Contraction                        | -2.560859 | 14      | 2     |
| Complement cascade                                 | -2.413080 | 18      | 8     |
| Platelet degranulation                             | -2.101032 | 70      | 9     |
| Response to elevated platelet cytosolic Ca2+       | -2.082864 | 74      | 9     |
| SARS-CoV Infections                                | 1.450543  | 255     | 88    |
| Signaling by Rho GTPases, Miro GTPases and RHOBTB3 | 1.545675  | 402     | 183   |
| Signaling by Rho GTPases                           | 1.549295  | 393     | 181   |
| RHO GTPase cycle                                   | 1.621215  | 278     | 96    |
| CDC42 GTPase cycle                                 | 1.632554  | 97      | 43    |
| RHOBTB2 GTPase cycle                               | 1.753969  | 16      | 7     |
| RHOB GTPase cycle                                  | 1.757849  | 48      | 30    |
| RHOA GTPase cycle                                  | 1.784755  | 94      | 50    |
| RHOBTB1 GTPase cycle                               | 1.813043  | 16      | 8     |
| RND3 GTPase cycle                                  | 1.857320  | 30      | 16    |
| RHOBTB GTPase Cycle                                | 1.877992  | 25      | 12    |

## 4WPC

### DNA vaccine

| Description                                                  | NES       | setSize | Count |
| ------------------------------------------------------------ | --------- | ------- | ----- |
| Metabolism of porphyrins                                     | 2.068871  | 16      | 9     |
| L1CAM interactions                                           | 1.995875  | 56      | 21    |
| Heme biosynthesis                                            | 1.961682  | 11      | 6     |
| Mitochondrial iron-sulfur cluster biogenesis                 | 1.877300  | 11      | 2     |
| The phototransduction cascade                                | 1.875759  | 11      | 2     |
| Inactivation, recovery and regulation of the phototransduction cascade | 1.875759  | 11      | 2     |
| Regulated Necrosis                                           | 1.839602  | 37      | 16    |
| Iron uptake and transport                                    | 1.771661  | 35      | 10    |
| Programmed Cell Death                                        | 1.726049  | 138     | 37    |
| Apoptosis                                                    | 1.607348  | 116     | 27    |
| Axon guidance                                                | 1.457884  | 318     | 80    |
| Metabolism of lipids                                         | -1.412703 | 395     | 100   |
| Diseases of metabolism                                       | -1.645383 | 108     | 42    |
| Diseases of glycosylation                                    | -1.701822 | 59      | 25    |
| Metabolism of steroids                                       | -1.725580 | 69      | 23    |
| Common Pathway of Fibrin Clot Formation                      | -1.871216 | 10      | 7     |
| Formation of Fibrin Clot (Clotting Cascade)                  | -1.932278 | 17      | 10    |
| Complement cascade                                           | -1.984173 | 20      | 5     |
| Regulation of Complement cascade                             | -2.004786 | 16      | 8     |
| Response to elevated platelet cytosolic Ca2+                 | -2.028299 | 80      | 17    |
| Insulin processing                                           | -2.043490 | 16      | 6     |
| Platelet degranulation                                       | -2.059360 | 76      | 17    |
| Post-translational protein phosphorylation                   | -2.217165 | 53      | 13    |
| Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs) | -2.225907 | 60      | 23    |

### EOMES

| Description                                                  | NES       | setSize | Count |
| ------------------------------------------------------------ | --------- | ------- | ----- |
| Metabolism of porphyrins                                     | 2.148982  | 16      | 8     |
| Heme biosynthesis                                            | 2.022551  | 11      | 6     |
| Keratinization                                               | 1.996972  | 17      | 4     |
| Formation of the cornified envelope                          | 1.996972  | 17      | 4     |
| CD209 (DC-SIGN) signaling                                    | 1.912676  | 14      | 5     |
| Generation of second messenger molecules                     | 1.906672  | 16      | 6     |
| Intrinsic Pathway for Apoptosis                              | 1.894452  | 31      | 12    |
| Cobalamin (Cbl, vitamin B12) transport and metabolism        | 1.838560  | 10      | 2     |
| Transcriptional regulation by the AP-2 (TFAP2) family of transcription factors | 1.802697  | 17      | 12    |
| The phototransduction cascade                                | 1.802022  | 11      | 4     |
| Inactivation, recovery and regulation of the phototransduction cascade | 1.802022  | 11      | 4     |
| Muscle contraction                                           | 1.779787  | 72      | 31    |
| Nonsense Mediated Decay (NMD) independent of the Exon Junction Complex (EJC) | 1.779769  | 75      | 47    |
| L1CAM interactions                                           | 1.758115  | 56      | 17    |
| Formation of a pool of free 40S subunits                     | 1.724827  | 83      | 50    |
| GTP hydrolysis and joining of the 60S ribosomal subunit      | 1.715622  | 92      | 53    |
| FCERI mediated Ca+2 mobilization                             | 1.709010  | 24      | 11    |
| Eukaryotic Translation Initiation                            | 1.708851  | 98      | 56    |
| Cap-dependent Translation Initiation                         | 1.708851  | 98      | 56    |
| Striated Muscle Contraction                                  | 1.706148  | 14      | 7     |
| L13a-mediated translational silencing of Ceruloplasmin expression | 1.695828  | 91      | 52    |
| Eukaryotic Translation Termination                           | 1.687454  | 73      | 45    |
| Eukaryotic Translation Elongation                            | 1.685674  | 72      | 53    |
| Peptide chain elongation                                     | 1.683744  | 70      | 44    |
| rRNA processing in the nucleus and cytosol                   | 1.679949  | 157     | 103   |
| Visual phototransduction                                     | 1.678981  | 29      | 5     |
| Estrogen-dependent gene expression                           | 1.660740  | 70      | 28    |
| Response of EIF2AK4 (GCN2) to amino acid deficiency          | 1.658338  | 81      | 50    |
| FOXO-mediated transcription                                  | 1.652512  | 36      | 19    |
| Major pathway of rRNA processing in the nucleolus and cytosol | 1.651185  | 147     | 95    |
| Selenoamino acid metabolism                                  | 1.649109  | 92      | 46    |
| Death Receptor Signaling                                     | 1.646330  | 95      | 38    |
| NR1H2 and NR1H3-mediated signaling                           | 1.639777  | 25      | 6     |
| Selenocysteine synthesis                                     | 1.636396  | 73      | 45    |
| Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell | 1.633773  | 25      | 11    |
| Nonsense-Mediated Decay (NMD)                                | 1.632377  | 89      | 51    |
| Nonsense Mediated Decay (NMD) enhanced by the Exon Junction Complex (EJC) | 1.632377  | 89      | 51    |
| rRNA processing                                              | 1.631788  | 165     | 106   |
| ESR-mediated signaling                                       | 1.589157  | 104     | 41    |
| Signaling by Nuclear Receptors                               | 1.503294  | 136     | 44    |
| Axon guidance                                                | 1.357830  | 318     | 70    |
| Asparagine N-linked glycosylation                            | -1.350545 | 190     | 87    |
| DNA Double-Strand Break Repair                               | -1.415577 | 90      | 30    |
| RHO GTPases Activate Formins                                 | -1.433743 | 88      | 22    |
| Diseases of metabolism                                       | -1.474885 | 108     | 42    |
| Cell Cycle, Mitotic                                          | -1.478818 | 359     | 80    |
| Homology Directed Repair                                     | -1.486808 | 71      | 26    |
| G2/M Checkpoints                                             | -1.497634 | 105     | 35    |
| M Phase                                                      | -1.511043 | 255     | 58    |
| DNA strand elongation                                        | -1.522111 | 28      | 11    |
| Cell Cycle                                                   | -1.534260 | 434     | 98    |
| Unfolded Protein Response (UPR)                              | -1.541637 | 70      | 27    |
| Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs) | -1.553914 | 60      | 22    |
| APC/C:Cdc20 mediated degradation of mitotic proteins         | -1.554022 | 63      | 27    |
| COPI-dependent Golgi-to-ER retrograde traffic                | -1.594287 | 59      | 21    |
| APC/C-mediated degradation of cell cycle proteins            | -1.594544 | 73      | 32    |
| Regulation of mitotic cell cycle                             | -1.594544 | 73      | 32    |
| Separation of Sister Chromatids                              | -1.600186 | 137     | 32    |
| Diseases of glycosylation                                    | -1.609731 | 59      | 20    |
| Activation of APC/C and APC/C:Cdc20 mediated degradation of mitotic proteins | -1.631899 | 64      | 28    |
| Telomere C-strand (Lagging Strand) Synthesis                 | -1.634018 | 24      | 12    |
| HDR through Homologous Recombination (HRR) or Single Strand Annealing (SSA) | -1.639731 | 66      | 25    |
| Mitotic Anaphase                                             | -1.644432 | 171     | 41    |
| Synthesis of very long-chain fatty acyl-CoAs                 | -1.670293 | 14      | 7     |
| Mitotic Metaphase and Anaphase                               | -1.679851 | 172     | 42    |
| Nuclear Pore Complex (NPC) Disassembly                       | -1.699523 | 29      | 11    |
| Resolution of D-Loop Structures                              | -1.710262 | 19      | 8     |
| Diseases of DNA repair                                       | -1.761204 | 32      | 16    |
| Mitotic Prometaphase                                         | -1.763232 | 132     | 39    |
| Golgi Cisternae Pericentriolar Stack Reorganization          | -1.766836 | 10      | 9     |
| Resolution of Sister Chromatid Cohesion                      | -1.777846 | 85      | 26    |
| Impaired BRCA2 binding to RAD51                              | -1.781533 | 23      | 10    |
| Cyclin A/B1/B2 associated events during G2/M transition      | -1.783024 | 16      | 8     |
| Phosphorylation of the APC/C                                 | -1.787385 | 15      | 6     |
| Resolution of D-loop Structures through Holliday Junction Intermediates | -1.790687 | 18      | 8     |
| HDR through Homologous Recombination (HRR)                   | -1.793318 | 42      | 17    |
| XBP1(S) activates chaperone genes                            | -1.798648 | 37      | 12    |
| Glycosaminoglycan metabolism                                 | -1.799723 | 58      | 17    |
| TP53 regulates transcription of several additional cell death genes whose specific roles in p53-dependent apoptosis remain uncertain | -1.801198 | 11      | 4     |
| Presynaptic phase of homologous DNA pairing and strand exchange | -1.803251 | 25      | 11    |
| G2/M DNA damage checkpoint                                   | -1.818246 | 42      | 15    |
| Resolution of D-loop Structures through Synthesis-Dependent Strand Annealing (SDSA) | -1.822299 | 15      | 8     |
| Diseases of DNA Double-Strand Break Repair                   | -1.828098 | 26      | 13    |
| Defective homologous recombination repair (HRR) due to BRCA2 loss of function | -1.828098 | 26      | 13    |
| Regulation of Complement cascade                             | -1.840737 | 16      | 9     |
| EML4 and NUDC in mitotic spindle formation                   | -1.851267 | 77      | 24    |
| Complement cascade                                           | -1.858660 | 20      | 11    |
| IRE1alpha activates chaperones                               | -1.859516 | 38      | 13    |
| Homologous DNA Pairing and Strand Exchange                   | -1.870689 | 27      | 14    |
| HDR through Single Strand Annealing (SSA)                    | -1.894345 | 26      | 12    |
| Diseases associated with glycosaminoglycan metabolism        | -1.925171 | 16      | 4     |
| Defective homologous recombination repair (HRR) due to BRCA1 loss of function | -1.931827 | 14      | 8     |
| Defective homologous recombination repair (HRR) due to PALB2 loss of function | -1.931827 | 14      | 8     |
| Defective HDR through Homologous Recombination Repair (HRR) due to PALB2 loss of BRCA1 binding function | -1.931827 | 14      | 8     |
| Defective HDR through Homologous Recombination Repair (HRR) due to PALB2 loss of BRCA2/RAD51/RAD51C binding function | -1.931827 | 14      | 8     |
| Impaired BRCA2 binding to PALB2                              | -1.931827 | 14      | 8     |
| Cell Cycle Checkpoints                                       | -1.995867 | 192     | 49    |
| Mitotic Spindle Checkpoint                                   | -2.074298 | 80      | 25    |
| Amplification of signal from the kinetochores                | -2.078635 | 68      | 22    |
| Amplification  of signal from unattached  kinetochores via a MAD2  inhibitory signal | -2.078635 | 68      | 22    |

### GATA3

| Description                                                  | NES       | setSize | Count |
| ------------------------------------------------------------ | --------- | ------- | ----- |
| Metabolism of porphyrins                                     | 2.202361  | 16      | 9     |
| Heme biosynthesis                                            | 2.074192  | 11      | 7     |
| Unfolded Protein Response (UPR)                              | -1.716388 | 70      | 26    |
| Glycosaminoglycan metabolism                                 | -1.740617 | 57      | 23    |
| Post-translational protein phosphorylation                   | -1.781209 | 50      | 19    |
| Sensory processing of sound                                  | -1.877618 | 25      | 5     |
| Plasma lipoprotein remodeling                                | -1.948396 | 16      | 5     |
| Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs) | -1.950295 | 57      | 18    |
| XBP1(S) activates chaperone genes                            | -2.023765 | 37      | 11    |
| IRE1alpha activates chaperones                               | -2.034111 | 38      | 13    |

### IV-HD

| Description                                                  | NES       | setSize | Count |
| ------------------------------------------------------------ | --------- | ------- | ----- |
| Metabolism of porphyrins                                     | 2.025030  | 17      | 9     |
| Heme biosynthesis                                            | 1.956092  | 11      | 6     |
| CD209 (DC-SIGN) signaling                                    | 1.905132  | 16      | 4     |
| Keratinization                                               | 1.866491  | 20      | 5     |
| Formation of the cornified envelope                          | 1.866491  | 20      | 5     |
| rRNA modification in the nucleus and cytosol                 | 1.790658  | 55      | 34    |
| rRNA processing in the nucleus and cytosol                   | 1.698858  | 160     | 84    |
| Major pathway of rRNA processing in the nucleolus and cytosol | 1.655784  | 150     | 77    |
| rRNA processing                                              | 1.612426  | 169     | 86    |
| Platelet activation, signaling and aggregation               | -1.792341 | 169     | 34    |
| XBP1(S) activates chaperone genes                            | -1.853295 | 39      | 16    |
| Sensory processing of sound by inner hair cells of the cochlea | -1.864196 | 25      | 8     |
| IRE1alpha activates chaperones                               | -1.908621 | 40      | 17    |
| Cholesterol biosynthesis                                     | -1.928222 | 19      | 13    |
| Post-translational protein phosphorylation                   | -2.089087 | 56      | 11    |
| Intrinsic Pathway of Fibrin Clot Formation                   | -2.117373 | 11      | 5     |
| Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs) | -2.172118 | 63      | 13    |
| Response to elevated platelet cytosolic Ca2+                 | -2.179590 | 85      | 25    |
| Common Pathway of Fibrin Clot Formation                      | -2.191657 | 10      | 8     |
| Platelet degranulation                                       | -2.214696 | 81      | 25    |
| Regulation of Complement cascade                             | -2.402575 | 19      | 8     |
| Formation of Fibrin Clot (Clotting Cascade)                  | -2.411237 | 19      | 10    |
| Complement cascade                                           | -2.440427 | 23      | 9     |

### IV-LD

| Description                                                  | NES       | setSize | Count |
| ------------------------------------------------------------ | --------- | ------- | ----- |
| Striated Muscle Contraction                                  | 2.125201  | 18      | 7     |
| Muscle contraction                                           | 2.011388  | 87      | 24    |
| Heme biosynthesis                                            | 1.951132  | 11      | 7     |
| Metabolism of porphyrins                                     | 1.946705  | 17      | 9     |
| Transcription of E2F targets under negative control by DREAM complex | 1.837145  | 16      | 9     |
| G0 and Early G1                                              | 1.825592  | 23      | 13    |
| TP53 Regulates Transcription of Cell Death Genes             | 1.822671  | 25      | 9     |
| Aberrant regulation of mitotic cell cycle due to RB1 defects | 1.819579  | 28      | 13    |
| Formation of tubulin folding intermediates by CCT/TriC       | 1.800396  | 13      | 8     |
| snRNP Assembly                                               | 1.797102  | 42      | 19    |
| Metabolism of non-coding RNA                                 | 1.797102  | 42      | 19    |
| Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell | 1.791607  | 27      | 12    |
| HCMV Early Events                                            | 1.775093  | 50      | 19    |
| Prefoldin mediated transfer of substrate  to CCT/TriC        | 1.771623  | 19      | 12    |
| Cooperation of Prefoldin and TriC/CCT  in actin and tubulin folding | 1.771623  | 19      | 12    |
| Transport of the SLBP Dependant Mature mRNA                  | 1.763561  | 27      | 14    |
| Transport of the SLBP independent Mature mRNA                | 1.760120  | 26      | 14    |
| Diseases of mitotic cell cycle                               | 1.756363  | 30      | 10    |
| tRNA processing in the nucleus                               | 1.745351  | 45      | 19    |
| Polymerase switching                                         | 1.737348  | 12      | 6     |
| Leading Strand Synthesis                                     | 1.737348  | 12      | 6     |
| Transcription of E2F targets under negative control by p107 (RBL1) and p130 (RBL2) in complex with HDAC1 | 1.727841  | 14      | 9     |
| SUMOylation of SUMOylation proteins                          | 1.706052  | 27      | 13    |
| Extension of Telomeres                                       | 1.699124  | 41      | 18    |
| S Phase                                                      | 1.697397  | 142     | 49    |
| tRNA processing                                              | 1.694374  | 85      | 38    |
| Synthesis of DNA                                             | 1.684453  | 105     | 54    |
| Switching of origins to a post-replicative state             | 1.661922  | 80      | 42    |
| CDK-mediated phosphorylation and removal of Cdc6             | 1.641596  | 63      | 31    |
| APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1 | 1.628545  | 64      | 32    |
| Mitochondrial protein degradation                            | 1.616809  | 80      | 33    |
| Cdc20:Phospho-APC/C mediated degradation of Cyclin A         | 1.599286  | 63      | 35    |
| Respiratory electron transport                               | 1.566409  | 82      | 31    |
| Factors involved in megakaryocyte development and platelet production | 1.549917  | 95      | 28    |
| G1/S Transition                                              | 1.540709  | 112     | 57    |
| Separation of Sister Chromatids                              | 1.508940  | 144     | 44    |
| Mitotic G1 phase and G1/S transition                         | 1.505042  | 123     | 42    |
| Aerobic respiration and respiratory electron transport       | 1.459795  | 157     | 57    |
| Metabolism of lipids                                         | -1.545778 | 490     | 129   |
| Metabolism of vitamins and cofactors                         | -1.624853 | 131     | 36    |
| Diseases of hemostasis                                       | -1.672322 | 11      | 5     |
| Response to elevated platelet cytosolic Ca2+                 | -1.708887 | 88      | 21    |
| ABC transporters in lipid homeostasis                        | -1.723269 | 10      | 4     |
| Post-translational protein phosphorylation                   | -1.725481 | 56      | 12    |
| Retinoid metabolism and transport                            | -1.731038 | 22      | 8     |
| Platelet degranulation                                       | -1.741427 | 84      | 21    |
| Diseases of Immune System                                    | -1.772153 | 14      | 4     |
| Diseases associated with the TLR signaling cascade           | -1.772153 | 14      | 4     |
| IRS-related events triggered by IGF1R                        | -1.773082 | 25      | 9     |
| Sensory processing of sound by outer hair cells of the cochlea | -1.775328 | 22      | 10    |
| Signaling by Type 1 Insulin-like Growth Factor 1 Receptor (IGF1R) | -1.786118 | 26      | 9     |
| IGF1R signaling cascade                                      | -1.786118 | 26      | 9     |
| Metabolism of fat-soluble vitamins                           | -1.795160 | 26      | 10    |
| Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs) | -1.801806 | 63      | 14    |
| Intrinsic Pathway of Fibrin Clot Formation                   | -1.845032 | 12      | 6     |
| Common Pathway of Fibrin Clot Formation                      | -1.897044 | 10      | 6     |
| Formation of Fibrin Clot (Clotting Cascade)                  | -2.077895 | 20      | 10    |
| Regulation of Complement cascade                             | -2.146441 | 21      | 10    |
| Complement cascade                                           | -2.181194 | 25      | 11    |

Need to do spleen 6WPC, liver (10WPI and 4WPC), and head-kidney (10WPI and 4WPC).

Scripts in use are *vaccine_contrasts.R*, *deseq_modelling_lymphoid.R*, and *gsea_new_and_definitive_4wpc.R*.