# Heart histoscore

![histoscore_4wpc_plot](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/histoscore_4wpc_plot.png)





# Differential Gene Expression

## Spleen

![dge_spleen_4wpc](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/spleen/results_4wpc/dge_spleen_4wpc.png)



### DNA vaccine

| Description - KEGG pathways | NES |
| --- | --- |
| Pancreatic secretion |  2.367240 |
| Protein digestion and absorption |  2.325324 |
| Porphyrin metabolism |  2.149408 |
| Cytoskeleton in muscle cells |  2.002579 |
| Salivary secretion |  1.886012 |
| Focal adhesion |  1.689051 |
| Complement and coagulation cascades | -2.361871 |







| Description - Reactome pathways | NES |
| --- | --- |
| Metabolism of porphyrins |  2.041732 |
| L1CAM interactions |  1.983006 |
| Heme biosynthesis |  1.887920 |
| Regulated Necrosis |  1.823170 |
| Programmed Cell Death |  1.724804 |
| **Response to elevated platelet cytosolic Ca2+** | -2.032585 |
| **Regulation of Complement cascade** | -2.023894 |
| **Complement cascade** | -2.003662 |
| Formation of Fibrin Clot (Clotting Cascade) | -1.950883 |
| Common Pathway of Fibrin Clot Formation | -1.894049 |
| Metabolism of steroids | -1.749787 |
| Diseases of glycosylation | -1.698180 |
| Diseases of metabolism | -1.639341 |
| Platelet activation, signaling and aggregation | -1.461187 |
| Metabolism of lipids | -1.403834 |





![dnavaccine_complement_cascade](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/spleen/results_4wpc/pathways/reactome/dnavaccine_complement_cascade.png)

<center> Figure x: Complement cascade pathway for DNA vaccine, obtained from Reactome (downregulated) </center>







![dnavaccine_regulation_complement_cascade](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/spleen/results_4wpc/pathways/reactome/dnavaccine_regulation_complement_cascade.png)

<center> Figure x: Regulation of Complement cascade pathway for DNA vaccine, obtained from Reactome (downregulated) </center>



### IV-LD

| Description - KEGG pathways | NES |
| --- | --- |
| Cytoskeleton in muscle cells |  1.892939 |
| Amyotrophic lateral sclerosis |  1.771305 |
| **Complement and coagulation cascades** | -2.439976 |



![hsa04610.ivld_complement_and_coagulation_cascades](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/spleen/results_4wpc/pathways/kegg/hsa04610.ivld_complement_and_coagulation_cascades.png)





| Description - Reactome pathways | NES |
| --- | --- |
| Striated Muscle Contraction |  2.155301 |
| Muscle contraction |  2.023175 |
| Heme biosynthesis |  1.981986 |
| Metabolism of porphyrins |  1.961239 |
| Formation of tubulin folding intermediates by CCT/TriC |  1.860691 |
| Aberrant regulation of mitotic cell cycle due to RB1 defects |  1.849127 |
| Transcription of E2F targets under negative control by DREAM complex |  1.845179 |
| G0 and Early G1 |  1.845082 |
| TP53 Regulates Transcription of Cell Death Genes |  1.842359 |
| snRNP Assembly |  1.815218 |
| Metabolism of non-coding RNA |  1.815218 |
| Signal attenuation | -1.713610 |
| Cholesterol biosynthesis | -1.696611 |
| Diseases of hemostasis | -1.693464 |
| Metal ion SLC transporters | -1.678358 |
| Plasma lipoprotein remodeling | -1.669208 |
| Amyloid fiber formation | -1.639151 |
| Metabolism of vitamins and cofactors | -1.634597 |
| Metabolism of lipids | -1.547511 |
| Regulation of lipid metabolism by PPARalpha | -1.518205 |
| RHOA GTPase cycle | -1.484069 |





![ivld_complement_cascade](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/spleen/results_4wpc/pathways/reactome/ivld_complement_cascade.png)

<center> Figure x: Complement cascade pathway in IV-LD, obtained from Reactome (downregulated) </center>









![ivld_regulation_complement_cascade](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/spleen/results_4wpc/pathways/reactome/ivld_regulation_complement_cascade.png)

<center> Figure x: Regulation of Complement cascade pathway for IV-LD, obtained from Reactome (downregulated) </center>



### IV-HD



| Description - KEGG pathways | NES |
| --- | --- |
| Pancreatic secretion |  2.559847 |
| Protein digestion and absorption |  2.452515 |
| Cell adhesion molecules |  2.054431 |
| Viral myocarditis |  2.026935 |
| Cytoskeleton in muscle cells |  1.902555 |
| Coronavirus disease - COVID-19 | -1.659684 |
| Protein processing in endoplasmic reticulum | -1.820332 |
| **Complement and coagulation cascades** | -2.659415 |







| Description - Reactome pathways | NES |
| --- | --- |
| Metabolism of porphyrins |  1.988530 |
| Heme biosynthesis |  1.961333 |
| rRNA modification in the nucleus and cytosol |  1.799053 |
| rRNA processing in the nucleus and cytosol |  1.710751 |
| Major pathway of rRNA processing in the nucleolus and cytosol |  1.661998 |
| rRNA processing |  1.618868 |
| Cholesterol biosynthesis | -1.894777 |
| Post-translational protein phosphorylation | -2.081798 |
| Intrinsic Pathway of Fibrin Clot Formation | -2.086623 |
| Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs) | -2.157572 |
| Common Pathway of Fibrin Clot Formation | -2.175846 |
| **Response to elevated platelet cytosolic Ca2+** | -2.183822 |
| Platelet degranulation | -2.208056 |
| **Regulation of Complement cascade** | -2.360902 |
| Formation of Fibrin Clot (Clotting Cascade) | -2.369414 |
| **Complement cascade** | -2.403786 |





### EOMES



| Description - KEGG pathways | NES |
| --- | --- |
| Pancreatic secretion | 2.439425 |
| Protein digestion and absorption | 2.332028 |
| Porphyrin metabolism | 2.245546 |
| Viral myocarditis | 1.979453 |
| Cytoskeleton in muscle cells | 1.847328 |



| Description - Reactome pathways | NES |
| --- | --- |
| Metabolism of porphyrins |  2.160501 |
| Keratinization |  2.031151 |
| Formation of the cornified envelope |  2.031151 |
| Heme biosynthesis |  2.020367 |
| rRNA processing in the nucleus and cytosol |  1.698060 |
| Mitotic Anaphase | -1.630522 |
| Mitotic Metaphase and Anaphase | -1.666687 |
| Mitotic Prometaphase | -1.734103 |
| Resolution of Sister Chromatid Cohesion | -1.802828 |
| EML4 and NUDC in mitotic spindle formation | -1.866674 |
| IRE1alpha activates chaperones | -1.893464 |
| Cell Cycle Checkpoints | -1.985594 |
| Amplification of signal from the kinetochores | -2.065563 |
| Amplification  of signal from unattached  kinetochores via a MAD2  inhibitory signal | -2.065563 |
| Mitotic Spindle Checkpoint | -2.079909 |







### GATA3

| Description - KEGG pathways | NES |
| --- | --- |
| Pancreatic secretion |  2.353118 |
| Porphyrin metabolism |  2.290375 |
| Protein digestion and absorption |  2.250693 |
| Protein processing in endoplasmic reticulum | -1.891300 |





| Description - Reactome pathways | NES |
| --- | --- |
| Metabolism of porphyrins |  2.221749 |
| Heme biosynthesis |  2.081954 |
| Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs) | -1.952083 |
| XBP1(S) activates chaperone genes | -2.020168 |
| IRE1alpha activates chaperones | -2.037904 |



## Liver



![deg_liver_4wpc](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/liver/results_4wpc/deg_liver_4wpc.png)







### DNA vaccine

| Description - KEGG pathways | NES |
| --- | --- |
| Tyrosine metabolism |  2.224872 |
| Ribosome |  2.140128 |
| Oxidative phosphorylation |  2.102454 |
| Drug metabolism - cytochrome P450 |  2.084978 |
| Fatty acid elongation |  2.074049 |
| Non-alcoholic fatty liver disease |  1.821710 |
| Thermogenesis |  1.541992 |
| Parkinson disease |  1.496866 |
| Huntington disease |  1.387719 |
| MAPK signaling pathway | -1.571250 |
| Efferocytosis | -1.635498 |
| Pathways in cancer | -1.657306 |
| Transcriptional misregulation in cancer | -1.673446 |
| Gastric cancer | -1.674566 |
| Leukocyte transendothelial migration | -1.755072 |
| Osteoclast differentiation | -1.757821 |
| Hippo signaling pathway | -1.786313 |
| Leishmaniasis | -1.819016 |
| Hematopoietic cell lineage | -1.879434 |



| Description - Reactome pathways | NES |
| --- | --- |
| SRP-dependent cotranslational protein targeting to membrane |  2.714197 |
| Eukaryotic Translation Elongation |  2.557913 |
| **Viral mRNA Translation** |  2.547726 |
| Peptide chain elongation |  2.511266 |
| Selenocysteine synthesis |  2.474216 |
| Nonsense Mediated Decay (NMD) independent of the Exon Junction Complex (EJC) |  2.471017 |
| Formation of a pool of free 40S subunits |  2.396173 |
| GTP hydrolysis and joining of the 60S ribosomal subunit |  2.381811 |
| L13a-mediated translational silencing of Ceruloplasmin expression |  2.381068 |
| Eukaryotic Translation Termination |  2.237764 |
| Hemostasis | -1.507290 |
| Platelet activation, signaling and aggregation | -1.532501 |
| Cell surface interactions at the vascular wall | -1.766200 |
| Cyclin D associated events in G1 | -1.768251 |
| G1 Phase | -1.768251 |
| **RAF-independent MAPK1/3 activation** | -1.798710 |
| Amino acid transport across the plasma membrane | -1.820184 |
| **Interleukin-4 and Interleukin-13 signaling** | -1.843352 |





### IV-LD



![hsa04062.ivld_Chemokine_signaling_pathway](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/liver/results_4wpc/pathways/kegg/hsa04062.ivld_Chemokine_signaling_pathway_downregulated.png)




| Description - KEGG pathways | NES |
| --- | --- |
| Ribosome |  2.214522 |
| Folate biosynthesis |  1.903833 |
| Alanine, aspartate and glutamate metabolism |  1.862574 |
| Oxidative phosphorylation |  1.810929 |
| **Platelet activation** | -1.791437 |
| **Phagosome** | -1.791703 |
| Gap junction | -1.810631 |
| Pathways in cancer | -1.815345 |
| Oxytocin signaling pathway | -1.843582 |
| **Chemokine signaling pathway** | -1.851650 |
| Leishmaniasis | -1.864045 |
| Long-term depression | -1.932296 |
| Osteoclast differentiation | -1.969536 |
| Hematopoietic cell lineage | -2.009696 |

​	More immune-related pathways besides the bottom 10 (downregulated):

- Th1 and Th2 cell differentiation
- Fc gamma R-mediated phagocytosis
- IL-17 signaling pathway
- Neutrophil extracellular trap formation
- Inflammatory mediator regulation of TRP channels
- Leukocyte transendothelial migration
- MAPK signaling pathway



| Description | NES |
| --- | --- |
| Formation of a pool of free 40S subunits |  2.461581 |
| **Viral mRNA Translation** |  2.442238 |
| Eukaryotic Translation Elongation |  2.441425 |
| L13a-mediated translational silencing of Ceruloplasmin expression |  2.404542 |
| Peptide chain elongation |  2.402242 |
| GTP hydrolysis and joining of the 60S ribosomal subunit |  2.398263 |
| Selenocysteine synthesis |  2.373498 |
| Nonsense Mediated Decay (NMD) independent of the Exon Junction Complex (EJC) |  2.324648 |
| Eukaryotic Translation Initiation |  2.306301 |
| Cap-dependent Translation Initiation |  2.306301 |
| Cell surface interactions at the vascular wall | -1.813760 |
| Signaling by ERBB2 in Cancer | -1.821919 |
| Signaling by ERBB2 KD Mutants | -1.821919 |
| Transport of inorganic cations/anions and amino acids/oligopeptides | -1.831634 |
| Amino acid transport across the plasma membrane | -1.856562 |
| **Fcgamma receptor (FCGR) dependent phagocytosis** | -1.859065 |
| Downregulation of SMAD2/3:SMAD4 transcriptional activity | -1.888862 |
| RHO GTPases Activate NADPH Oxidases | -1.896570 |
| Transcriptional activity of SMAD2/SMAD3:SMAD4 heterotrimer | -1.908295 |
| **Antigen activates B Cell Receptor (BCR) leading to generation of second messengers** | -1.928643 |



​	More immune-related pathways besides the bottom 10 (downregulated):

- Platelet activation, signaling and aggregation
- FCGR3A-mediated IL10 synthesis
- ROS and RNS production in phagocytes
- Signaling by Interleukins
- Signaling by Receptor Tyrosine Kinases
- Diseases of signal transduction by growth factor receptors and second messengers
- Cytokine Signaling in Immune system
- Adaptive Immune System



### IV-HD





No KEGG pathways were enriched.

| Description - Reactome pathways | NES |
| --- | --- |
| FOXO-mediated transcription |  2.136423 |
| FOXO-mediated transcription of oxidative stress, metabolic and neuronal genes |  1.952877 |
| Cell Cycle | -1.615394 |
| Cell Cycle, Mitotic | -1.728346 |
| Synthesis of DNA | -1.801086 |
| rRNA modification in the nucleus and cytosol | -1.928438 |
| **Interleukin-4 and Interleukin-13 signaling** | -2.041756 |
| Activation of the pre-replicative complex | -2.079796 |
| tRNA processing | -2.119945 |
| tRNA modification in the nucleus and cytosol | -2.203471 |







![ivhd_Interleukin-4_and_Interleukin-13_signaling](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/liver/results_4wpc/pathways/reactome/ivhd_Interleukin-4_and_Interleukin-13_signaling.png)



<center> Figure x: Interleukin-4 and Interleukin-13 signaling pathway for IV-HD, obtained from Reactome (downregulated) </center>



### EOMES



| Description - KEGG pathways | NES |
| --- | --- |
| Steroid biosynthesis |  2.225854 |
| Biosynthesis of unsaturated fatty acids |  2.176682 |
| Ribosome |  2.138680 |
| Drug metabolism - cytochrome P450 |  2.067329 |
| Butanoate metabolism |  2.007449 |
| Retinol metabolism |  1.951916 |
| Fat digestion and absorption |  1.951315 |
| Fatty acid elongation |  1.938097 |
| Biosynthesis of nucleotide sugars |  1.937404 |
| Biosynthesis of cofactors |  1.906382 |
| B cell receptor signaling pathway | -1.969320 |
| Rap1 signaling pathway | -1.976913 |
| Leukocyte transendothelial migration | -1.982056 |
| Pathways in cancer | -1.983536 |
| ECM-receptor interaction | -1.991998 |
| Chemokine signaling pathway | -1.998556 |
| Leishmaniasis | -2.003777 |
| Fc gamma R-mediated phagocytosis | -2.019878 |
| Osteoclast differentiation | -2.079215 |
| Platelet activation | -2.199653 |



More immune-related pathways besides the bottom 10 (downregulated):

- IL-17 signaling pathway
- C-type lectin receptor signaling pathway
- Fc epsilon RI signaling pathway
- Th17 cell differentiation
- Natural killer cell mediated cytotoxicity
- NOD-like receptor signaling pathway
- Th1 and Th2 cell differentiation
- Neutrophil extracellular trap formation
- JAK-STAT signaling pathway
- NF-kappa B signaling pathway
- Toll-like receptor signaling pathway
- MAPK signaling pathway



![hsa04662.eomes_B_cell_receptor_signaling_pathway](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/liver/results_4wpc/pathways/kegg/hsa04662.eomes_B_cell_receptor_signaling_pathway.png)





| Description - Reactome pathways | NES |
| --- | --- |
| Cholesterol biosynthesis |  2.555243 |
| SRP-dependent cotranslational protein targeting to membrane |  2.410313 |
| Activation of gene expression by SREBF (SREBP) |  2.400503 |
| Regulation of cholesterol biosynthesis by SREBP (SREBF) |  2.325864 |
| Metabolism of steroids |  2.298777 |
| Translation |  2.198763 |
| Formation of a pool of free 40S subunits |  2.180123 |
| Viral mRNA Translation |  2.101684 |
| Selenocysteine synthesis |  2.055102 |
| GTP hydrolysis and joining of the 60S ribosomal subunit |  2.025638 |
| GPVI-mediated activation cascade | -1.999874 |
| RHO GTPase Effectors | -2.018252 |
| Cell surface interactions at the vascular wall | -2.022804 |
| G alpha (s) signalling events | -2.038391 |
| Platelet Aggregation (Plug Formation) | -2.048564 |
| Integrin signaling | -2.070343 |
| **FCGR3A-mediated IL10 synthesis** | -2.085934 |
| GPER1 signaling | -2.087874 |
| **Fcgamma receptor (FCGR) dependent phagocytosis** | -2.098381 |
| **Interleukin-4 and Interleukin-13 signaling** | -2.274045 |



More immune-related pathways besides the bottom 10 (downregulated):

- Interferon gamma signaling 
- Signaling by CSF1 (M-CSF) in myeloid cells
- FCGR3A-mediated phagocytosis
- Platelet activation, signaling and aggregation
-  Interleukin-3, Interleukin-5 and GM-CSF signaling
- Toll Like Receptor 7/8 (TLR7/8) Cascade
- MyD88 dependent cascade initiated on endosome
- Cytokine Signaling in Immune system
- Toll Like Receptor 9 (TLR9) Cascade
- Signaling by Interleukins
- TRAF6 mediated induction of NFkB and MAP kinases upon TLR7/8 or 9 activation
- MyD88-independent TLR4
- TRIF (TICAM1)-mediated TLR4 signaling 
- Toll Like Receptor 4 (TLR4)
- Toll Like Receptor 3 (TLR3) Cascade         





![eomes_Interleukin-4_and_Interleukin-13_signaling](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/liver/results_4wpc/pathways/reactome/eomes_Interleukin-4_and_Interleukin-13_signaling.png)



<center> Figure x: Interleukin-4 and Interleukin-13 signaling pathway for EOMES, obtained from Reactome (downregulated) </center>



### GATA3

| Description - KEGG pathways | NES |
| --- | --- |
| Retinol metabolism |  2.298595 |
| beta-Alanine metabolism |  2.141258 |
| Oxidative phosphorylation |  2.100320 |
| Alanine, aspartate and glutamate metabolism |  2.020371 |
| Tyrosine metabolism |  2.008027 |
| Maturity onset diabetes of the young |  1.986700 |
| Parkinson disease |  1.969279 |
| Valine, leucine and isoleucine degradation |  1.896997 |
| Glyoxylate and dicarboxylate metabolism |  1.864732 |
| Amyotrophic lateral sclerosis |  1.848949 |
| Non-alcoholic fatty liver disease |  1.843825 |
| Biosynthesis of cofactors |  1.817478 |
| Alzheimer disease |  1.508261 |
| Huntington disease |  1.476799 |
| Pathways of neurodegeneration - multiple diseases |  1.449477 |
| PI3K-Akt signaling pathway | -1.596141 |
| Transcriptional misregulation in cancer | -1.666039 |
| Pathways in cancer | -1.671421 |
| Chemokine signaling pathway | -1.693283 |
| ECM-receptor interaction | -1.855409 |
| Hematopoietic cell lineage | -2.166258 |



![hsa04062.gata3_chemokine_signaling_pathway](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/liver/results_4wpc/pathways/kegg/hsa04062.gata3_chemokine_signaling_pathway.png)





| Description - Reactome pathways | NES |
| --- | --- |
| The canonical retinoid cycle in rods (twilight vision) |  2.340552 |
| Aerobic respiration and respiratory electron transport |  1.726151 |
| Metabolism of amino acids and derivatives |  1.516465 |
| Hemostasis | -1.611026 |
| **GPCR downstream signalling** | -1.708258 |
| **Signaling by GPCR** | -1.710773 |
| Cell surface interactions at the vascular wall | -1.845194 |
| Transport of inorganic cations/anions and amino acids/oligopeptides | -1.924938 |







## Head-kidney



![deg_hkidney_4wpc](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/hkidney/results_4wpc/deg_hkidney_4wpc.png)







### DNA vaccine

| Description - KEGG pathways | NES |
| --- | --- |
| Parkinson disease |  2.694081 |
| Oxidative phosphorylation |  2.692693 |
| Ribosome |  2.608588 |
| Prion disease |  2.516970 |
| Diabetic cardiomyopathy |  2.428673 |
| Amyotrophic lateral sclerosis |  2.425394 |
| Proteasome |  2.412838 |
| Huntington disease |  2.409665 |
| Chemical carcinogenesis - reactive oxygen species |  2.313707 |
| Non-alcoholic fatty liver disease |  2.005124 |
| Coronavirus disease - COVID-19 |  1.954962 |
| Pathways of neurodegeneration - multiple diseases |  1.947185 |
| Alzheimer disease |  1.944298 |
| Thermogenesis |  1.905752 |
| Calcium signaling pathway | -1.600536 |
| cGMP-PKG signaling pathway | -1.603060 |
| Gastric cancer | -1.640748 |
| Endometrial cancer | -1.696137 |
| Colorectal cancer | -1.699318 |
| Neuroactive ligand-receptor interaction | -1.718577 |
| Breast cancer | -1.864058 |
| Basal cell carcinoma | -2.020670 |





| Description - Reactome pathways | NES |
| --- | --- |
| Regulation of expression of SLITs and ROBOs |  2.776747 |
| Mitochondrial protein degradation |  2.695600 |
| GTP hydrolysis and joining of the 60S ribosomal subunit |  2.685938 |
| Formation of a pool of free 40S subunits |  2.667244 |
| L13a-mediated translational silencing of Ceruloplasmin expression |  2.666095 |
| Eukaryotic Translation Initiation |  2.638982 |
| Cap-dependent Translation Initiation |  2.638982 |
| SRP-dependent cotranslational protein targeting to membrane |  2.614994 |
| Signaling by ROBO receptors |  2.611671 |
| Viral mRNA Translation |  2.586357 |
| Class A/1 (Rhodopsin-like receptors) | -1.667166 |
| RHOB GTPase cycle | -1.718850 |
| **NRAGE signals death through JNK** | -1.774065 |
| Defective B3GALT6 causes EDSP2 and SEMDJL1 | -1.817820 |
| GPCR downstream signalling | -1.824973 |
| Signaling by GPCR | -1.874780 |
| G alpha (12/13) signalling events | -1.909884 |



More immune-related pathways besides the top 10 (upregulated):

- Response to EIF2AK4 (GCN2) to amino acid deficiency: EIF2AK4 is involved in the integrated stress response, which can affect immune cell survival and function under conditions of nutrient stress

- Cross-presentation of soluble exogenous antigens (endosomes)
- Regulation of RUNX3 expression and activity - RUNX3 plays a role in the differentiation and function of immune cells, including cytotoxic T lymphocytes and dendritic cells
- Vif-mediated degradation of APOBEC3G
- Regulation of Apoptosis
- Adaptive Immune System







![dnavaccine_NRAGE_signals_death_through_JNK](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/hkidney/results_4wpc/pathways/reactome/dnavaccine_NRAGE_signals_death_through_JNK.png)

<center> Figure x: NRAGE signals death through NK pathway for DNA vaccine, obtained from Reactome (downregulated) </center>

> NRAGE signals death through JNK - The JNK pathway is involved in cellular stress responses, inflammation, and apoptosis, playing a role in immune regulation and inflammation. (ChatGPT)







### IV-LD



| Description - KEGG pathways | NES |
| --- | --- |
| Diabetic cardiomyopathy | 1.426588 |



​	No Reactome pathways enriched at either p < 0.05 or p < 0.1.





### IV-HD



| Description | NES |
| --- | --- |
| Parkinson disease | 2.308131 |
| Porphyrin metabolism | 2.243616 |
| Ribosome | 2.203357 |
| Proteasome | 2.128755 |
| Prion disease | 2.073043 |
| Huntington disease | 2.049222 |
| **Viral protein interaction with cytokine and cytokine receptor** | 2.021708 |
| Tryptophan metabolism | 1.923582 |
| Amyotrophic lateral sclerosis | 1.920960 |
| Oxidative phosphorylation | 1.918434 |
| DNA replication | 1.860834 |
| Diabetic cardiomyopathy | 1.854227 |
| Alzheimer disease | 1.745848 |
| Biosynthesis of cofactors | 1.737939 |
| Coronavirus disease - COVID-19 | 1.701898 |
| Pathways of neurodegeneration - multiple diseases | 1.692541 |
| Chemical carcinogenesis - reactive oxygen species | 1.603999 |





<img src="/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/hkidney/results_4wpc/pathways/kegg/hsa04061.ivhd_viral_protein_interaction_with_cytokine_and_cytokine_receptor.png" alt="hsa04061.ivhd_viral_protein_interaction_with_cytokine_and_cytokine_receptor" style="zoom:150%;" />





| Description - Reactome pathways | NES |
| --- | --- |
| GTP hydrolysis and joining of the 60S ribosomal subunit |  2.709539 |
| Regulation of expression of SLITs and ROBOs |  2.699474 |
| Formation of a pool of free 40S subunits |  2.680630 |
| **L13a-mediated translational silencing of Ceruloplasmin expression** |  2.669082 |
| Eukaryotic Translation Initiation |  2.631062 |
| Cap-dependent Translation Initiation |  2.631062 |
| Eukaryotic Translation Elongation |  2.576302 |
| **Response of EIF2AK4 (GCN2) to amino acid deficiency** |  2.574874 |
| Peptide chain elongation |  2.569430 |
| Nonsense Mediated Decay (NMD) independent of the Exon Junction Complex (EJC) |  2.547706 |
| Spry regulation of FGF signaling | -1.768561 |
| Negative regulation of FGFR1 signaling | -1.768561 |
| Negative regulation of FGFR3 signaling | -1.768561 |
| Negative regulation of FGFR4 signaling | -1.768561 |
| Chondroitin sulfate/dermatan sulfate metabolism | -1.784021 |
| FLT3 signaling in disease | -1.812055 |
| IRE1alpha activates chaperones | -1.824537 |
| XBP1(S) activates chaperone genes | -1.852217 |
| **Interleukin-6 family signaling** | -1.956013 |
| **Signaling by CSF1 (M-CSF) in myeloid cells** | -2.005930 |



More immune-related pathways besides the top 10 (upregulated):

- Cross-presentation of soluble exogenous antigens (endosomes): Pathway involved in presenting exogenous antigens to T cells through endocytic pathways.

- Vif-mediated degradation of APOBEC3G: This pathway describes the mechanism by which the viral protein Vif targets APOBEC3G, an antiviral factor, for degradation.

- Vpu mediated degradation of CD4: Describes how the viral protein Vpu facilitates the degradation of CD4, a surface receptor on T cells, to evade immune detection.

- NIK–>noncanonical NF-kB signaling: Involves the noncanonical NF-kB signaling pathway regulated by NIK, important for immune responses and inflammation.

- Dectin-1 mediated noncanonical NF-kB signaling: This pathway involves Dectin-1 receptor-mediated activation of noncanonical NF-kB signaling, crucial for antifungal and immune responses.

> Pathway descriptions from ChatGPT.







![ivhd_Interleukin-6_family_signaling](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/hkidney/results_4wpc/pathways/reactome/ivhd_Interleukin-6_family_signaling.png)

<center> Figure x: Interleukin-6 family signaling family pathway for IV-HD, obtained from Reactome (downregulated) </center>







![ivhd_Signaling_by_CSF1_(M-CSF)_in_myeloid_cells](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/hkidney/results_4wpc/pathways/reactome/ivhd_Signaling_by_CSF1_(M-CSF)_in_myeloid_cells.png)

<center> Figure x: Signaling by CSF1 (M-CSF) in myeloid cells pathway for IV-HD, obtained from Reactome (downregulated) </center>



### EOMES



| Description - KEGG pathways | NES |
| --- | --- |
| Ribosome |  2.549684 |
| Oxidative phosphorylation |  2.505681 |
| Diabetic cardiomyopathy |  2.487711 |
| Carbon metabolism |  2.465706 |
| Parkinson disease |  2.386492 |
| Chemical carcinogenesis - reactive oxygen species |  2.239810 |
| Prion disease |  2.184492 |
| Amyotrophic lateral sclerosis |  2.116883 |
| Huntington disease |  2.098237 |
| Alzheimer disease |  1.870719 |
| Pathways of neurodegeneration - multiple diseases |  1.715044 |
| Polycomb repressive complex | -1.751485 |





| Description - Reactome pathways | NES |
| --- | --- |
| Formation of a pool of free 40S subunits |  2.808401 |
| GTP hydrolysis and joining of the 60S ribosomal subunit |  2.798870 |
| L13a-mediated translational silencing of Ceruloplasmin expression |  2.773856 |
| Eukaryotic Translation Initiation |  2.758928 |
| Cap-dependent Translation Initiation |  2.758928 |
| **Viral mRNA Translation** |  2.757048 |
| Eukaryotic Translation Elongation |  2.752745 |
| Metabolism of amino acids and derivatives |  2.732790 |
| **Response of EIF2AK4 (GCN2) to amino acid deficiency** |  2.729296 |
| Peptide chain elongation |  2.720672 |
| GPCR downstream signalling | -1.568531 |
| Signaling by GPCR | -1.592451 |







### GATA3

| Description - KEGG pathways | NES |
| --- | --- |
| Parkinson disease | 2.371020 |
| Ribosome | 2.289597 |
| Oxidative phosphorylation | 2.247578 |
| Prion disease | 2.126575 |
| Diabetic cardiomyopathy | 2.111010 |
| Chemical carcinogenesis - reactive oxygen species | 2.048196 |
| Huntington disease | 2.009283 |
| Amyotrophic lateral sclerosis | 1.987158 |
| Alzheimer disease | 1.767541 |
| Pathways of neurodegeneration - multiple diseases | 1.743515 |



| Description - Reactome pathways | NES |
| --- | --- |
| Maternal to zygotic transition (MZT) | 2.711458 |
| Regulation of expression of SLITs and ROBOs | 2.620981 |
| Formation of a pool of free 40S subunits | 2.599283 |
| GTP hydrolysis and joining of the 60S ribosomal subunit | 2.596564 |
| L13a-mediated translational silencing of Ceruloplasmin expression | 2.570332 |
| Eukaryotic Translation Initiation | 2.526143 |
| Cap-dependent Translation Initiation | 2.526143 |
| Signaling by ROBO receptors | 2.525509 |
| Eukaryotic Translation Elongation | 2.500757 |
| Nonsense Mediated Decay (NMD) independent of the Exon Junction Complex (EJC) | 2.493719 |





## Heart

### DNA vaccine

| Description - KEGG pathways | NES |
| --- | --- |
| **Oxidative phosphorylation** |  2.253219 |
| Thermogenesis |  2.177964 |
| Chemical carcinogenesis - reactive oxygen species |  2.141827 |
| Non-alcoholic fatty liver disease |  2.084876 |
| Parkinson disease |  2.084711 |
| Diabetic cardiomyopathy |  2.059580 |
| Circadian rhythm |  2.055741 |
| Huntington disease |  2.051598 |
| Amyotrophic lateral sclerosis |  1.971876 |
| 2-Oxocarboxylic acid metabolism |  1.952746 |
| Epstein-Barr virus infection | -1.730101 |
| Osteoclast differentiation | -1.734971 |
| Sphingolipid metabolism | -1.741227 |
| **Natural killer cell mediated cytotoxicity** | -1.746715 |
| **Antigen processing and presentation** | -1.764198 |
| **PD-L1 expression and PD-1 checkpoint pathway in cancer** | -1.807417 |
| **NF-kappa B signaling pathway** | -1.810737 |
| Cytosolic DNA-sensing pathway | -1.940597 |
| **B cell receptor signaling pathway** | -2.048872 |
| **Primary immunodeficiency** | -2.165443 |

Other immune-related pathways (downregulated):

- Phagosome: Involved in engulfing and breaking down pathogens, key for innate immune responses.

- Fc gamma R-mediated phagocytosis: Enhances immune defense by recognizing antibodies and initiating phagocytosis of pathogens.

- T cell receptor signaling pathway: Central to adaptive immunity, activating T cells for pathogen recognition and response 

- NOD-like receptor signaling pathway: Detects intracellular pathogens and triggers immune responses, including inflammation.

- Th17 cell differentiation: Involves differentiation of T helper cells that produce pro-inflammatory cytokines.



![hsa04662.B_cell_receptor_signaling_pathway_downregulated](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/heart/results_4wpc/pathways/kegg/hsa04662.dnavaccine_B_cell_receptor_signaling_pathway_downregulated.png)




| Description - Reactome pathways | NES |
| --- | --- |
| Aerobic respiration and respiratory electron transport |  2.546779 |
| Respiratory electron transport, ATP synthesis by chemiosmotic coupling, and heat production by uncoupling proteins. |  2.536069 |
| Respiratory electron transport |  2.467103 |
| Complex I biogenesis |  2.249453 |
| Mitochondrial protein degradation |  2.101075 |
| Mitochondrial biogenesis |  2.087504 |
| Cristae formation |  2.058220 |
| Branched-chain amino acid catabolism |  2.027424 |
| Formation of ATP by chemiosmotic coupling |  1.977119 |
| Citric acid cycle (TCA cycle) |  1.970660 |
| **FCERI mediated MAPK activation** | -1.951362 |
| Cytosolic tRNA aminoacylation | -1.967269 |
| **Signaling by the B Cell Receptor (BCR)** | -2.068112 |
| **Antigen activates B Cell Receptor (BCR) leading to generation of second messengers** | -2.080046 |
| RHO GTPases Activate NADPH Oxidases | -2.096529 |
| GPVI-mediated activation cascade | -2.151865 |
| **Antigen processing-Cross presentation** | -2.168779 |
| **Interferon alpha/beta signaling** | -2.246796 |
| **Signaling by CSF1 (M-CSF) in myeloid cells** | -2.252415 |
| ROS and RNS production in phagocytes | -2.274398 |



Other immune-related pathways (downregulated):

- Interleukin-3, Interleukin-5 and GM-CSF signaling: Regulates the growth, differentiation, and function of blood cells, enhancing immune responses.

- ER-Phagosome pathway: Involved in antigen processing and presentation to initiate immune responses.

- Diseases of Immune System: Refers to disorders that affect the immune system, leading to immune deficiency or autoimmunity.

- Diseases associated with the TLR signaling cascade: Involves Toll-like receptors that detect pathogens and trigger immune responses, linked to inflammatory diseases.

- FCGR3A-mediated phagocytosis: Involves the recognition of antibody-coated pathogens, promoting their clearance by immune cells.

- Antigen Presentation: Folding, assembly and peptide loading of class I MHC: Essential for presenting antigens to T cells to trigger adaptive immune responses.

- Activation of NF-kappaB in B cells: Promotes immune signaling and the development of B cell immune responses.

- Regulation of actin dynamics for phagocytic cup formation: Controls actin structure during the engulfment of pathogens by phagocytes.

- Fcgamma receptor (FCGR) dependent phagocytosis: Mediates immune cell recognition and ingestion of antibody-tagged pathogens.

- TCR signaling: Activates T cells through the T cell receptor, critical for adaptive immune responses.

- SARS-CoV-2 activates/modulates innate and adaptive immune responses: Describes how COVID-19 alters both innate and adaptive immunity.

- Fc epsilon receptor (FCERI) signaling: Plays a role in allergic responses by triggering mast cell and basophil activation.

- C-type lectin receptors (CLRs): Recognize pathogens, triggering immune cell activation and cytokine production.

- Interleukin-1 family signaling: Mediates inflammatory responses, often involved in fever and acute phase reactions.

- Neutrophil degranulation: Involves the release of antimicrobial compounds by neutrophils to combat infections.

- Cytokine Signaling in Immune system: Involves the transmission of signals by cytokines to regulate immune responses.

- Signaling by Interleukins: Refers to the activation of immune cells by various interleukins, crucial for immune modulation.

- Adaptive Immune System: Involves specific, long-lasting immune responses through B and T lymphocytes.



![dnavaccine_Antigen_activates_B_Cell_Receptor_(BCR)_leading_to_generation_of_second_messengers](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/heart/results_4wpc/pathways/reactome/dnavaccine_Antigen_activates_B_Cell_Receptor_(BCR)_leading_to_generation_of_second_messengers.png)

<center> Figure x: Antigen activates B Cell Receptor (BCR) leading to generation of second messengers pathway for DNA vaccine, obtained from Reactome (downregulated) </center>





### IV-LD




| Description - KEGG pathways | NES |
| --- | --- |
| 2-Oxocarboxylic acid metabolism |  2.149566 |
| Thermogenesis |  2.096084 |
| Oxidative phosphorylation |  2.090468 |
| Circadian rhythm |  2.009489 |
| Citrate cycle (TCA cycle) |  2.008406 |
| Lipoic acid metabolism |  1.997176 |
| Chemical carcinogenesis - reactive oxygen species |  1.992294 |
| Valine, leucine and isoleucine degradation |  1.989368 |
| Cardiac muscle contraction |  1.958234 |
| Diabetic cardiomyopathy |  1.952411 |
| Proteasome | -1.909864 |
| Osteoclast differentiation | -1.915884 |
| **Phagosome** | -1.949208 |
| Aminoacyl-tRNA biosynthesis | -1.963852 |
| Coronavirus disease - COVID-19 | -1.965134 |
| **B cell receptor signaling pathway** | -1.977919 |
| **Antigen processing and presentation** | -2.025796 |
| **Cytosolic DNA-sensing pathway** | -2.077620 |
| Primary immunodeficiency | -2.148789 |
| Lysosome | -2.217197 |



Other immune-related pathways (downregulated):

- Cytokine-cytokine receptor interaction: Key communication mechanism in immune responses. NES: -1.51

- Chemokine signaling pathway: Regulates the migration and activation of immune cells. NES: -1.54

- Toll-like receptor signaling pathway: Detects pathogens and activates innate immune responses. NES: -1.58

- T cell receptor signaling pathway: Critical for T cell activation and adaptive immunity. NES: -1.58

- Th17 cell differentiation: Pathway involved in the differentiation of pro-inflammatory Th17 cells. NES: -1.61

- Protein processing in endoplasmic reticulum: Involves the folding and modification of proteins crucial for immune function. NES: -1.65

- Epstein-Barr virus infection: Immune response to Epstein-Barr virus. NES: -1.66

- JAK-STAT signaling pathway: Regulates immune cell proliferation, differentiation, and activation. NES: -1.66

- Fc epsilon RI signaling pathway: Involved in allergic reactions and immune cell activation. NES: -1.66

- Leishmaniasis: Immune response to the parasite Leishmania. NES: -1.67

- RIG-I-like receptor signaling pathway: Detects viral RNA and triggers antiviral immune responses. NES: -1.70

- NOD-like receptor signaling pathway: Recognizes intracellular pathogens and triggers immune responses. NES: -1.70

- Hematopoietic cell lineage: Pathway regulating the development of immune cells from hematopoietic stem cells. NES: -1.70

- NF-kappa B signaling pathway: Central in controlling immune and inflammatory responses. NES: -1.73

- Fc gamma R-mediated phagocytosis: Promotes the uptake and destruction of pathogens by immune cells. NES: -1.75

- Natural killer cell mediated cytotoxicity: NK cells’ role in destroying infected or cancerous cells. NES: -1.78

- Leukocyte transendothelial migration: Guides immune cells to sites of infection or injury. NES: -1.82







![hsa04662.ivld_B_cell_receptor_signaling](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/heart/results_4wpc/pathways/kegg/hsa04662.ivld_B_cell_receptor_signaling.png)





![hsa04612.ivld_antigen_processing_and_presentation](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/heart/results_4wpc/pathways/kegg/hsa04612.ivld_antigen_processing_and_presentation.png)





| Description - Reactome pathways | NES |
| --- | --- |
| Aerobic respiration and respiratory electron transport |  2.454398 |
| Respiratory electron transport, ATP synthesis by chemiosmotic coupling, and heat production by uncoupling proteins. |  2.399751 |
| Mitochondrial biogenesis |  2.313920 |
| Respiratory electron transport |  2.304207 |
| Cristae formation |  2.121271 |
| Complex I biogenesis |  2.094107 |
| Formation of ATP by chemiosmotic coupling |  2.047924 |
| Citric acid cycle (TCA cycle) |  2.047638 |
| Mitochondrial protein degradation |  2.030663 |
| Lysine catabolism |  1.965286 |
| Hh mutants are degraded by ERAD | -2.137880 |
| Hh mutants abrogate ligand secretion | -2.137880 |
| Glycosphingolipid catabolism | -2.176635 |
| **Signaling by the B Cell Receptor (BCR)** | -2.185310 |
| RHO GTPases Activate NADPH Oxidases | -2.188485 |
| **Antigen activates B Cell Receptor (BCR) leading to generation of second messengers** | -2.263672 |
| **ROS and RNS production in phagocytes** | -2.336786 |
| Cytosolic tRNA aminoacylation | -2.339713 |
| **Interferon alpha/beta signaling** | -2.342496 |
| **Antigen processing-Cross presentation** | -2.371937 |



Other immune-related pathways (downregulated):

- Adaptive Immune System: Involves T and B cell-mediated immune responses. NES: -1.38
- Leishmania infection: Represents immune responses against Leishmania parasites. NES: -1.58
-  Parasitic Infection Pathways: Encompasses host-pathogen interactions during parasitic infections. NES: -1.58
-  CLEC7A (Dectin-1) signaling: Detects fungal pathogens and triggers immune responses. NES: -1.59
-  SARS-CoV-2 Infection: Involves the immune response to SARS-CoV-2 viral infection. NES: -1.59
-  Toll-like Receptor Cascades: Initiates immune responses by recognizing pathogen-associated molecular patterns. NES: -1.62
-  C-type lectin receptors (CLRs): Part of the innate immune system for pathogen detection. NES: -1.65
-  Signaling by Interleukins: Facilitates immune cell communication through interleukins. NES: -1.66
-  Downstream signaling events of B Cell Receptor (BCR): Activates B cells leading to antibody production. NES: -1.66
-  Downstream TCR signaling: Mediates T cell activation in adaptive immune responses. NES: -1.67
-  Interleukin-1 family signaling: Triggers immune and inflammatory responses via interleukin-1. NES: -1.74
-  Interleukin-1 signaling: Involves the regulation of immune and inflammatory responses. NES: -1.74
-  Interleukin-3, Interleukin-5, and GM-CSF signaling: Regulates the production and function of immune cells. NES: -1.74
-  Fc epsilon receptor (FCERI) signaling: Plays a role in allergic reactions and immune cell activation. NES: -1.75
-  Cytokine Signaling in Immune system: Mediates communication between immune cells through cytokines. NES: -1.76
-  TRAF6 mediated IRF7 activation: Activates antiviral immune responses through interferon signaling. NES: -1.81
-  Interferon Signaling: Manages antiviral defenses by activating immune cells. NES: -1.81
-  Interferon gamma signaling: Controls immune cell activation in viral and bacterial infections. NES: -1.82
-  TCR signaling: Essential for T cell activation in adaptive immunity. NES: -1.91
-  Cross-presentation of soluble exogenous antigens (endosomes): Presents antigens to T cells to initiate immune responses. NES: -1.87
-  FCERI mediated NF-kB activation: Activates NF-kB, promoting inflammation and immune defense. NES: -1.89
-  FCGR3A-mediated phagocytosis: Promotes phagocytosis of pathogens through immune cell receptors. NES: -1.98
-  Fcgamma receptor (FCGR) dependent phagocytosis: Mediates pathogen uptake and destruction by immune cells. NES: -1.96
-  Signaling by CSF1 (M-CSF) in myeloid cells: Regulates myeloid immune cell differentiation and function. NES: -1.99
-  Inactivation of CSF3 (G-CSF) signaling: Involves the regulation of neutrophil production. NES: -1.99
-  SARS-CoV-2 activates/modulates innate and adaptive immune responses: Refers to the immune response triggered by SARS-CoV-2. NES: -2.12







![ivld_Antigen_processing-Cross_presentation](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/heart/results_4wpc/pathways/reactome/ivld_Antigen_processing-Cross_presentation.png)



<center> Figure x: Antigen processing-Cross presentation pathway for IV-LD, obtained from Reactome (downregulated) </center>





![ivld_Interferon_alpha-beta_signaling](/Users/ffi007/Library/CloudStorage/OneDrive-UiTOffice365/Documents/PhD/Thesis/quantseq_dataAnalysis/deseq2_dataAnalysis_2024/results/heart/results_4wpc/pathways/reactome/ivld_Interferon_alpha-beta_signaling.png)



<center> Figure x: Interferon alpha-beta signaling pathway for IV-HD, obtained from Reactome (downregulated) </center>

### IV-HD



| Description - KEGG pathways | NES |
| --- | --- |
| Thermogenesis |  2.232249 |
| Oxidative phosphorylation |  2.231884 |
| Chemical carcinogenesis - reactive oxygen species |  2.177648 |
| Diabetic cardiomyopathy |  2.093991 |
| Non-alcoholic fatty liver disease |  2.034847 |
| 2-Oxocarboxylic acid metabolism |  2.018248 |
| Cardiac muscle contraction |  2.010906 |
| Tryptophan metabolism |  1.989889 |
| Parkinson disease |  1.968256 |
| Circadian rhythm |  1.958020 |
| Complement and coagulation cascades | -1.846306 |
| Biosynthesis of nucleotide sugars | -1.847286 |
| NOD-like receptor signaling pathway | -1.861411 |
| Steroid biosynthesis | -1.960784 |
| Phagosome | -1.983688 |
| Primary immunodeficiency | -1.987760 |
| Proteasome | -2.033015 |
| Leishmaniasis | -2.058419 |
| Antigen processing and presentation | -2.079315 |
| Lysosome | -2.090083 |

Other immune-related pathways (downregulated):

- Fc gamma R-mediated phagocytosis: Immune response involving receptor-mediated engulfment of pathogens. NES: -1.66

- Leukocyte transendothelial migration: Movement of immune cells across endothelial cells to infection sites. NES: -1.69

- Epstein-Barr virus infection: Viral infection affecting B-cell functions and associated with immune disorders. NES: -1.69

- Sphingolipid metabolism: Involves immune response regulation, inflammation, and apoptosis. NES: -1.71

- Osteoclast differentiation: Differentiation of bone-resorbing cells with immune system interactions. NES: -1.74

- Cytosolic DNA-sensing pathway: Immune mechanism for detecting foreign DNA in the cytosol. NES: -1.75

- B cell receptor signaling pathway: Activation of B cells during immune responses. NES: -1.78

- Staphylococcus aureus infection: Immune response to bacterial infection affecting multiple cell types. NES: -1.81

- Tuberculosis: Chronic bacterial infection causing strong immune system activation. NES: -1.82

- Coronavirus disease - COVID-19: Viral infection leading to diverse immune responses. NES: -1.83

- Amino sugar and nucleotide sugar metabolism: Metabolic pathway affecting immune responses. NES: -1.84

- Other glycan degradation: Breakdown of glycans impacting immune and metabolic processes. NES: -1.84





| Description - Reactome pathways | NES |
| --- | --- |
| Aerobic respiration and respiratory electron transport |  2.620556 |
| Respiratory electron transport, ATP synthesis by chemiosmotic coupling, and heat production by uncoupling proteins. |  2.595040 |
| Respiratory electron transport |  2.477497 |
| Mitochondrial biogenesis |  2.281283 |
| Cristae formation |  2.193866 |
| Complex I biogenesis |  2.189417 |
| Formation of ATP by chemiosmotic coupling |  2.145738 |
| Mitochondrial protein degradation |  2.136163 |
| Citric acid cycle (TCA cycle) |  2.103274 |
| Branched-chain amino acid catabolism |  1.946489 |
| G2/M Checkpoints | -2.102832 |
| ER-Phagosome pathway | -2.138566 |
| Hh mutants are degraded by ERAD | -2.152786 |
| Hh mutants abrogate ligand secretion | -2.152786 |
| Hedgehog ligand biogenesis | -2.160955 |
| Interferon alpha/beta signaling | -2.185847 |
| Glycosphingolipid catabolism | -2.235053 |
| Cholesterol biosynthesis | -2.252918 |
| **ROS and RNS production in phagocytes** | -2.348626 |
| **Antigen processing-Cross presentation** | -2.399970 |



Other immune-related pathways (downregulated):

- Toll-like Receptor Cascades: Pathways triggering immune response to pathogens. NES: -1.53
- Signaling by Interleukins: Cytokine signaling affecting immune system regulation. NES: -1.53
- Interferon Signaling: Cytokine response regulating antiviral defenses. NES: -1.55
- Cytokine Signaling in Immune system: Pathways modulating immune cell communication. NES: -1.59
- Fcgamma receptor (FCGR) dependent phagocytosis: Immune response through receptor-mediated pathogen engulfment. NES: -1.59
- Interleukin-1 signaling: Inflammatory cytokine signaling involved in immune response. NES: -1.67
- Interleukin-1 family signaling: Family of cytokines driving immune and inflammatory responses. NES: -1.68
- Interleukin-12 family signaling: Drives immune cell differentiation and response. NES: -1.70
- Fc epsilon receptor (FCERI) signaling: Involved in allergic reactions and immune response modulation. NES: -1.70
- Parasite infection: Immune response to parasitic pathogens. NES: -1.74
- Leishmania phagocytosis: Immune process for engulfing Leishmania parasites. NES: -1.74
- FCGR3A-mediated phagocytosis: Activates phagocytic immune response through specific receptors. NES: -1.74
- C-type lectin receptors (CLRs): Immune receptors recognizing carbohydrates on pathogens. NES: -1.74
- CLEC7A (Dectin-1) signaling: Pathway recognizing fungal pathogens and initiating immune responses. NES: -1.75
- Interferon gamma signaling: Activates immune cells to fight viral and bacterial infections. NES: -1.77
- SARS-CoV-2 activates/modulates innate and adaptive immune responses: Involves both early and adaptive immune responses to COVID-19. NES: -1.77
- Regulation of actin dynamics for phagocytic cup formation: Modulates actin for effective pathogen engulfment. NES: -1.78
- Signaling by CSF3 (G-CSF): Regulates immune cell production in response to infection. NES: -1.79
- DAP12 interactions: Signaling events that activate immune cells. NES: -1.80
- DAP12 signaling: Pathway involving immune cell activation and response. NES: -1.80
- TNFR2 non-canonical NF-kB pathway: Regulates immune and inflammatory responses via NF-kB signaling. NES: -1.80



### EOMES



| Description - KEGG pathways | NES |
| --- | --- |
| Thermogenesis | 2.002009 |





| Description - Reactome pathways | NES |
| --- | --- |
| Aerobic respiration and respiratory electron transport |  2.299705 |
| Respiratory electron transport, ATP synthesis by chemiosmotic coupling, and heat production by uncoupling proteins. |  2.287949 |
| Respiratory electron transport |  2.228167 |
| Mitochondrial biogenesis |  2.087083 |
| **Adaptive Immune System** | -1.482060 |
| **Cytokine Signaling in Immune system** | -1.608004 |
| **Neutrophil degranulation** | -1.684634 |





### GATA3

| Description - KEGG pathways | NES |
| --- | --- |
| Non-alcoholic fatty liver disease |  2.094880 |
| Chemical carcinogenesis - reactive oxygen species |  2.091920 |
| Thermogenesis |  2.084083 |
| Diabetic cardiomyopathy |  1.958517 |
| Oxidative phosphorylation |  1.941024 |
| Parkinson disease |  1.921509 |
| Circadian rhythm |  1.895900 |
| FoxO signaling pathway |  1.870052 |
| Ribosome |  1.866958 |
| Huntington disease |  1.850008 |
| **Lysosome** | -1.722111 |
| Amino sugar and nucleotide sugar metabolism | -1.746892 |
| **Leukocyte transendothelial migration** | -1.748348 |
| **Proteasome** | -1.795927 |
| Biosynthesis of nucleotide sugars | -1.835002 |
| **Fc gamma R-mediated phagocytosis** | -1.866387 |
| Aminoacyl-tRNA biosynthesis | -1.870241 |
| Collecting duct acid secretion | -1.970527 |
| **Complement and coagulation cascades** | -1.996630 |
| **B cell receptor signaling pathway** | -2.121911 |

Other immune-related pathways (downregulated):

- NOD-like receptor signaling pathway. NES: -1.54
- Yersinia infection. NES: -1.55
- Phagosome. NES: -1.62







| Description - Reactome pathways | NES |
| --- | --- |
| Aerobic respiration and respiratory electron transport |  2.537260 |
| Respiratory electron transport, ATP synthesis by chemiosmotic coupling, and heat production by uncoupling proteins. |  2.443850 |
| Respiratory electron transport |  2.329992 |
| Cristae formation |  2.154357 |
| Complex I biogenesis |  2.134986 |
| Mitochondrial biogenesis |  2.118488 |
| Peptide chain elongation |  2.077850 |
| Formation of ATP by chemiosmotic coupling |  2.070323 |
| Eukaryotic Translation Elongation |  2.048752 |
| Nonsense Mediated Decay (NMD) independent of the Exon Junction Complex (EJC) |  2.017815 |
| **Antigen activates B Cell Receptor (BCR) leading to generation of second messengers** | -2.077152 |
| Cytosolic tRNA aminoacylation | -2.143818 |
| GPVI-mediated activation cascade | -2.147769 |
| **RHO GTPases Activate NADPH Oxidases** | -2.161473 |
| **Signaling by CSF1 (M-CSF) in myeloid cells** | -2.180718 |
| **Diseases of Immune System** | -2.203874 |
| **Diseases associated with the TLR signaling cascade** | -2.203874 |
| **Antigen processing-Cross presentation** | -2.255048 |
| **ROS and RNS production in phagocytes** | -2.389562 |
| **Interferon alpha/beta signaling** | -2.442447 |



Other immune-related pathways (downregulated):

- Adaptive Immune System: Mechanisms underlying the body’s ability to recognize and respond to specific pathogens. NES: -1.42

- Host Interactions of HIV factors: Pathways affected by HIV interactions with host cells. NES: -1.59

- Interleukin-1 signaling: Inflammatory cytokine signaling involved in immune response. NES: -1.63

- Signaling by Interleukins: Cytokine signaling that regulates various aspects of immune responses. NES: -1.65

- Toll-like Receptor Cascades: Pathways that activate immune responses in response to pathogen-associated molecular patterns. NES: -1.67

- Fcgamma receptor (FCGR) dependent phagocytosis: Immune response through receptor-mediated engulfment of pathogens. NES: -1.69

- Cytokine Signaling in Immune system: Pathways that modulate communication between immune cells. NES: -1.70

- Neutrophil degranulation: Release of granules from neutrophils to combat infections. NES: -1.70

- Fc epsilon receptor (FCERI) signaling: Involved in allergic reactions and modulation of immune responses. NES: -1.71

- Interferon Signaling: Cytokine signaling that regulates antiviral immune responses. NES: -1.72

- Interleukin-1 family signaling: Family of cytokines driving immune and inflammatory responses. NES: -1.74

- Downstream TCR signaling: Signaling pathways activated following T-cell receptor engagement. NES: -1.75

- FCERI mediated NF-kB activation: Pathways involving the activation of NF-kB through Fc epsilon receptor signaling. NES: -1.77

- SARS-CoV-2 activates/modulates innate and adaptive immune responses: Impact of SARS-CoV-2 on both early and adaptive immune responses. NES: -1.87

- Parasite infection: Immune responses to parasitic pathogens. NES: -1.90

- Leishmania phagocytosis: Immune process for engulfing Leishmania parasites. NES: -1.90

- FCGR3A-mediated phagocytosis: Phagocytic immune response activated by FCGR3A receptor. NES: -1.90

- Interleukin-3, Interleukin-5 and GM-CSF signaling: Cytokine signaling involved in hematopoiesis and immune cell function. NES: -1.93

- Signaling by the B Cell Receptor (BCR): Pathways involved in B cell activation and response. NES: -2.03

  